<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
    <title>Document</title>
    <link rel="stylesheet" href="../themes/eLife/styles.css" />
  </head>

  <body>
    <article>
      <p>
        <a href="https://elifesciences.org/articles/45107">
          elifesciences.org
        </a>
      </p>

      <h1>
        Reciprocal action of Casein Kinase Iε on core planar polarity proteins
        regulates clustering and asymmetric localisation
      </h1>

      <h3>
        Authors of this article
      </h3>

      <ol>
        <li><a href="/articles/45107#xfac6e154">Helen Strutt</a>,</li>
        <li><a href="/articles/45107#xe0305c57">Jessica Gamage</a>,</li>
        <li>
          <a href="/articles/45107#x3840d38a">
            David Strutt
          </a>
        </li>
      </ol>

      <h3>
        Author institutions
      </h3>
      <ol>
        <li>
          University of Sheffield, United Kingdom;
        </li>
      </ol>

      <h2>
        Abstract
      </h2>

      <p>
        The conserved core planar polarity pathway is essential for coordinating
        polarised cell behaviours and the formation of polarised structures such
        as cilia and hairs. Core planar polarity proteins localise
        asymmetrically to opposite cell ends and form intercellular complexes
        that link the polarity of neighbouring cells. This asymmetric
        segregation is regulated by phosphorylation through poorly understood
        mechanisms. We show that loss of phosphorylation of the core protein
        Strabismus in the
        <em>
          Drosophila
        </em>
        pupal wing increases its stability and promotes its clustering at
        intercellular junctions, and that Prickle negatively regulates
        Strabismus phosphorylation. Additionally, loss of phosphorylation of
        Dishevelled – which normally localises to opposite cell edges to
        Strabismus – reduces its stability at junctions. Moreover, both
        phosphorylation events are independently mediated by Casein Kinase Iε.
        We conclude that Casein Kinase Iε phosphorylation acts as a switch,
        promoting Strabismus mobility and Dishevelled immobility, thus enhancing
        sorting of these proteins to opposite cell edges.
      </p>

      <pre
        class="language-js line-numbers"
        data-line="2"
      ><code>const x = () => {
  console.log('welcome!')
}</code></pre>

      <p>
        <a href="https://doi.org/10.7554/eLife.45107.001">
          https://doi.org/10.7554/eLife.45107.001
        </a>
      </p>

      <h2>
        Introduction
      </h2>

      <p>
        Phosphorylation is a widespread means of controlling protein activity,
        regulating protein-protein interactions, protein stability and
        conformation (
        <a href="#bib26">
          Hunter, 2007
        </a>
        ). The activity of most signalling pathways is regulated by
        phosphorylation of pathway components. This includes the ‘core’ planar
        polarity pathway (
        <a href="#bib44">
          Singh and Mlodzik, 2012
        </a>
        ;
        <a href="#bib13">
          Butler and Wallingford, 2017
        </a>
        ): however, compared to other signalling pathways, the molecular
        mechanisms are poorly understood.
      </p>

      <pre class="language-html line-numbers"><code>&lt;head&gt;
  &lt;meta charset=&quot;UTF-8&quot; /&gt;
  &lt;meta name=&quot;viewport&quot; content=&quot;width=device-width, initial-scale=1.0&quot; /&gt;
  &lt;meta http-equiv=&quot;X-UA-Compatible&quot; content=&quot;ie=edge&quot; /&gt;
  &lt;title&gt;Document&lt;/title&gt;
  &lt;link rel=&quot;stylesheet&quot; href=&quot;../themes/eLife/stencila-theme.css&quot; /&gt;
&lt;/head&gt;</code></pre>

      <p>
        The core planar polarity proteins (hereafter, the ‘core proteins’)
        regulate the production of polarised structures or polarised cell
        behaviours in the plane of a tissue. This includes polarised production
        of cilia and of stereocilia bundles in the inner ear, and the
        coordinated polarisation of tissue movements necessary for convergence
        and extension of the body axis (
        <a href="#bib20">
          Devenport, 2016
        </a>
        ;
        <a href="#bib19">
          Davey and Moens, 2017
        </a>
        ;
        <a href="#bib13">
          Butler and Wallingford, 2017
        </a>
        ). In
        <em>
          Drosophila
        </em>
        , the core pathway controls the production of polarised hairs and
        bristles on many adult tissues, for example the trichomes that emerge
        from the distal edge of each cell in the adult wing.
      </p>
      <p>
        The core pathway specifies polarised structures via the asymmetric
        localisation of pathway components. In the
        <em>
          Drosophila
        </em>
        pupal wing, the seven-pass transmembrane protein Frizzled (Fz), and the
        cytoplasmic proteins Dishevelled (Dsh) and Diego (Dgo) localise to
        distal cell ends, where the trichome will emerge. The four-pass
        transmembrane protein Strabismus (Stbm, also known as Van Gogh [Vang])
        and Prickle (Pk) localise to proximal cell ends, and the atypical
        cadherin Flamingo (Fmi, also known as Starry Night [Stan]) localises to
        both proximal and distal cell ends (
        <a href="#fig1">
          Figure 1A
        </a>
        ). Fmi mediates homophilic adhesion that is important for coupling
        polarity between cells (reviewed in
        <a href="#bib24">
          Goodrich and Strutt, 2011
        </a>
        ;
        <a href="#bib20">
          Devenport, 2016
        </a>
        ;
        <a href="#bib13">
          Butler and Wallingford, 2017
        </a>
        ).
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig1-v2.tif/full/1500,/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig1-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Planar polarity and the cloud model of core protein localisation.
          </h6>
          <p>
            (
            <strong>
              A
            </strong>
            ) Core polarity proteins localise to proximal or distal edges of
            pupal wing cells (left), where they form intercellular complexes
            (right). (
            <strong>
              B
            </strong>
            ) Live image of a 28 hr APF pupal wing expressing …
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.002">
            https://doi.org/10.7554/eLife.45107.002
          </a>
        </figcaption>
      </figure>

      <p>
        The overall direction of polarisation is determined by tissue-specific
        global cues (
        <a href="#bib3">
          Aw and Devenport, 2017
        </a>
        ). Polarity is then thought to be refined and amplified by feedback
        interactions between the core proteins. Mathematical modelling has
        suggested that feedback may involve destabilisation of complexes of
        opposite orientation and/or stabilisation of complexes in the same
        orientation. This can lead to sorting of complexes such that they all
        align in the same direction (
        <a href="#bib2">
          Amonlirdviman et al., 2005
        </a>
        ;
        <a href="#bib31">
          Le Garrec et al., 2006
        </a>
        ;
        <a href="#bib11">
          Burak and Shraiman, 2009
        </a>
        ;
        <a href="#bib40">
          Schamberg et al., 2010
        </a>
        ).
      </p>
      <p>
        With regard to possible stabilising mechanisms, core protein asymmetry
        is associated with clustering of proteins into punctate membrane
        subdomains (
        <a href="#fig1">
          Figure 1B
        </a>
        ,
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ;
        <a href="#bib16">
          Cho et al., 2015
        </a>
        ) and reduced core protein turnover (
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ;
        <a href="#bib12">
          Butler and Wallingford, 2015
        </a>
        ;
        <a href="#bib15">
          Chien et al., 2015
        </a>
        ;
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). Based on a detailed study of core protein organisation in puncta, we
        recently proposed that core proteins form a non-stoichiometric ‘cloud’
        around a Fmi-Fz nucleus (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). Feedback interactions lead to sorting of complexes, and multiple
        protein-protein interactions are thought to promote a phase transition
        into higher order ‘signalosome-like’ structures, where arrays of
        complexes of the same orientation are stabilised (
        <a href="#fig1">
          Figure 1C
        </a>
        ,
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). Interestingly, Stbm stoichiometry was found to be much higher than
        that of the other core proteins (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). The reasons for this are unclear, but could relate to a role for Stbm
        in promoting higher order structures. Furthermore, Pk may stabilise Stbm
        by promoting complex clustering (
        <a href="#bib52">
          Tree et al., 2002
        </a>
        ;
        <a href="#bib6">
          Bastock et al., 2003
        </a>
        ;
        <a href="#bib16">
          Cho et al., 2015
        </a>
        ).
      </p>
      <p>
        Mechanisms of destabilisation may include competitive binding between
        core proteins (
        <a href="#bib52">
          Tree et al., 2002
        </a>
        ;
        <a href="#bib14">
          Carreira-Barbosa et al., 2003
        </a>
        ;
        <a href="#bib28">
          Jenny et al., 2005
        </a>
        ;
        <a href="#bib2">
          Amonlirdviman et al., 2005
        </a>
        ). More specifically, Pk (a ‘proximal’ complex component) is known to
        destabilise Fz and/or Dsh (‘distal’ components) in the same cell (
        <a href="#bib55">
          Warrington et al., 2017
        </a>
        ). In addition, Pk has been suggested to destabilise complexes
        containing Stbm and Fmi (
        <a href="#bib16">
          Cho et al., 2015
        </a>
        ). However, knowledge of additional molecular mechanisms by which core
        proteins might become destabilised or clustered together is very poor,
        and post-translational modifications such as phosphorylation are likely
        to be a key element.
      </p>
      <p>
        Indeed, core protein phosphorylation is essential for feedback
        amplification of asymmetry. In particular, reduced activity of Casein
        Kinase Iε (CKIε, also known as Discs Overgrown [Dco] or Doubletime [Dbt]
        in flies) causes planar polarity defects and a reduction in core protein
        asymmetry (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ;
        <a href="#bib30">
          Klein et al., 2006
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). Interestingly, CKIε has been implicated in phosphorylation of both
        Stbm and Dsh. CKIε was first found to bind to and phosphorylate the
        vertebrate Dsh homologue (Dvl) in canonical Wnt signalling (
        <a href="#bib38">
          Peters et al., 1999
        </a>
        ;
        <a href="#bib32">
          McKay et al., 2001
        </a>
        ). In planar polarity in flies, Dsh phosphorylation correlates with its
        recruitment to cellular junctions by Fz (
        <a href="#bib5">
          Axelrod, 2001
        </a>
        ;
        <a href="#bib43">
          Shimada et al., 2001
        </a>
        ), where it is incorporated into stable complexes (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ), and decreased Dsh phosphorylation is seen in
        <em>
          dco
        </em>
        mutants (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ).
      </p>
      <p>
        The exact phosphorylation sites for CKIε in Dsh/Dvl are not well
        defined, but a mutation of a serine/threonine-rich region upstream of
        the PDZ domain affects Dvl recruitment to membranes in
        <em>
          Xenopus
        </em>
        (
        <a href="#bib33">
          Ossipova et al., 2005
        </a>
        ). Moreover, mutation of one of these residues (S236 in fly Dsh) blocks
        phosphorylation of Dsh by Dco in vitro (
        <a href="#bib30">
          Klein et al., 2006
        </a>
        ). However, a transgene in which these residues were mutated largely
        rescued the planar polarity defects of
        <em>
          dsh
        </em>
        mutants in the adult fly wing (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ; but see also
        <a href="#bib36">
          Penton et al., 2002
        </a>
        ).
      </p>
      <p>
        More recently, CKIε has been implicated in phosphorylating Stbm and its
        vertebrate homologue Vangl2 (
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). In particular, Wnt gradients were proposed to lead to a gradient of
        Vangl2 phosphorylation and asymmetry in the vertebrate limb (
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ). CKIε promotes Stbm/Vangl2 phosphorylation in cell culture (
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). Two clusters of conserved serine and threonine residues were
        identified as CKIε phosphorylation sites. Mutation of some or all
        of these residues leads to a loss of Stbm/Vangl2 phosphorylation in cell
        culture, and defects in planar polarisation (
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib34">
          Ossipova et al., 2015
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ).
      </p>
      <p>
        The fact that CKIε has been implicated in phosphorylating both
        Stbm/Vangl2 and Dsh/Dvl in cell culture leads to the question of whether
        both proteins are bona fide targets in vivo. For instance, both Fz and
        Dsh/Dvl have been proposed to promote Stbm/Vangl2 phosphorylation by
        CKIε (
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). Thus, it is possible that only Stbm/Vangl2 are direct targets of CKIε
        and that Stbm/Vangl2 phosphorylation has a secondary effect on
        Fz-Dsh/Dvl behaviour. Moreover, mechanistic insight into how these
        phosphorylation events affect core protein sorting and asymmetry is
        lacking.
      </p>
      <p>
        Here, we demonstrate that CKIε has independent and reciprocal actions on
        Dsh and Stbm during planar polarity signalling in
        <em>
          Drosophila
        </em>
        . We use phosphorylation site mutations in Stbm to show that lack of
        Stbm phosphorylation leads to its clustering in ‘mixed’ puncta that
        contain complexes in both orientations. CKIε-dependent phosphorylation
        increases Stbm turnover at junctions, and thus promotes complex sorting,
        while phosphorylation of Dsh decreases its turnover. Pk negatively
        regulates Stbm phosphorylation and increases Stbm stability. These
        results support a direct role for Dco in phosphorylating both Stbm and
        Dsh in vivo in planar polarity signalling.
      </p>

      <h2>
        Results
      </h2>

      <h3>
        Stbm phosphorylation sites are essential for core protein asymmetry
      </h3>

      <p>
        Previous work identified two conserved clusters of serine and threonine
        residues within vertebrate Vangl2, which are phosphorylated in tissue
        culture (
        <a href="#fig1">
          Figure 1D
        </a>
        ,
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ).
        <em>
          P[acman]-stbm
        </em>
        rescue constructs (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ) were generated, in which all serine/threonine residues in clusters I
        and II were mutated to alanine (phosphomutant ‘S[All]A’) or glutamic
        acid (phosphomimetic ‘S[All]E’). These residues are in regions of Stbm
        predicted to be unstructured (data not shown), so this was not expected
        to alter the secondary structure of Stbm. Neither the phosphomutant nor
        the phosphomimetic form of Stbm rescued the trichome orientation of
        <em>
          stbm
        </em>
        null mutants, while wild-type
        <em>
          P[acman]-stbm
        </em>
        in the same genomic site gave complete rescue (
        <a href="#fig2">
          Figure 2A–D
        </a>
        and
        <a href="https://elifesciences.org/articles/45107/figures#fig2s1">
          Figure 2—figure supplement 1D
        </a>
        ). The failure of the phosphomimetic version to rescue may be because
        glutamic acid does not completely substitute for phosphorylated serine
        and threonine residues within Stbm. Alternatively, Stbm may need to
        cycle between phosphorylated and unphosphorylated forms in order to
        function in planar polarity, and the mutated proteins are unable to
        perform this cycling.
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig2-v2.tif/full/,1500/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig2-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Disrupted trichome orientation in Stbm phosphomutants and
            phosphomimetics.
          </h6>
          <p>
            (
            <strong>
              A–D
            </strong>
            ) Adult wings from
            <em>
              stbm
              <sup>
                6
              </sup>
            </em>
            mutant flies (
            <strong>
              A
            </strong>
            ),
            <em>
              P[acman]-stbm stbm
              <sup>
                6
              </sup>
            </em>
            flies (
            <strong>
              B
            </strong>
            ),
            <em>
              P[acman]-stbmS[All]A stbm
              <sup>
                6
              </sup>
            </em>
            flies (
            <strong>
              C
            </strong>
            ), or
            <em>
              P[acman]-stbmS[All]E stbm
              <sup>
                6
              </sup>
            </em>
            flies (
            <strong>
              D
            </strong>
            ). Scale bar 100 µm. (
            <strong>
              E,F
            </strong>
            ) 28 hr APF pupal …
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.003">
            https://doi.org/10.7554/eLife.45107.003
          </a>
        </figcaption>
      </figure>

      <p>
        Core protein localisation in phosphomutant and phosphomimetic pupal
        wings was then examined. Twin clones were made, in which tissue
        expressing wild-type Stbm was juxtaposed to tissue expressing mutant
        forms of Stbm, both in the absence of endogenous
        <em>
          stbm
        </em>
        gene activity. In keeping with the strong trichome orientation defects,
        a strong decrease in core protein asymmetry was seen in pupal wings
        expressing either phosphomutant or phosphomimetic forms of Stbm (
        <a href="#fig2">
          Figure 2E–H
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2s1">
          Figure 2—figure supplement 1H and I
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ). There was also a slight increase in overall levels of phosphomutant
        or phosphomimetic Stbm at cellular junctions, compared to wild-type Stbm
        (
        <a href="#fig2">
          Figure 2I
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ). We conclude that the phosphorylation sites in Stbm are necessary for
        its correct asymmetric localisation and to orient trichomes in the adult
        wing.
      </p>
      <p>
        Western blotting of pupal wing extracts confirmed that endogenous Stbm
        is phosphorylated in vivo, with the majority of protein existing in a
        phosphorylated state (
        <a href="https://elifesciences.org/articles/45107/figures#fig2s2">
          Figure 2—figure supplement 2A
        </a>
        ). As expected, the phosphomutant form showed increased mobility on
        SDS-PAGE, while the phosphomimetic form had a similar mobility to that
        of endogenous Stbm (
        <a href="#fig2">
          Figure 2J
        </a>
        ). However, as expected, the mobility of phosphomimetic Stbm was not
        sensitive to phosphatase treatment (
        <a href="https://elifesciences.org/articles/45107/figures#fig2s2">
          Figure 2—figure supplement 2A
        </a>
        ). Overall cellular protein levels were similar to wild-type (
        <a href="#fig2">
          Figure 2K
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ).
      </p>
      <p>
        It has been suggested that residue 5 in cluster II, and residues 120 and
        122 in cluster I are ‘founder sites’, such that phosphorylation on these
        leads to a cascade of phosphorylation on neighbouring residues (
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib34">
          Ossipova et al., 2015
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). However,
        <em>
          P[acman]-stbm
        </em>
        constructs simultaneously mutant for all three founder sites fully
        rescued trichome polarity in adult wings, and core protein asymmetry in
        pupal wings was normal (
        <a href="https://elifesciences.org/articles/45107/figures#fig2s1">
          Figure 2—figure supplement 1
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ). This contrasts with the work of
        <a href="#bib29">
          Kelly et al. (2016)
        </a>
        , who reported planar polarity defects in flies after mutation of
        serines 120 and 122 to alanine. The difference in our results could be
        due to abnormal or uneven expression of the
        <em>
          tub-StbmS2A
        </em>
        rescue construct used in
        <a href="#bib29">
          Kelly et al. (2016)
        </a>
        . Furthermore, mutation of all phosphorylation sites in only cluster I
        or cluster II revealed that the phosphorylation sites within cluster I
        are sufficient for correct core protein asymmetry, and are responsible
        for most of the retardation in mobility on SDS-PAGE (
        <a href="https://elifesciences.org/articles/45107/figures#fig2s2">
          Figure 2—figure supplement 2B–H
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ).
      </p>

      <h3>
        Uncoupling of puncta formation and asymmetry in Stbm phosphomutants
      </h3>

      <p>
        Interestingly, previous data has suggested a correlation between core
        protein asymmetric localisation to opposite cell ends, and the formation
        of large junctional puncta (
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ;
        <a href="#bib16">
          Cho et al., 2015
        </a>
        ). In keeping with this idea, fewer puncta were observed in Stbm
        phosphomimetic flies, and non-puncta material also increased (
        <a href="#fig2">
          Figure 2F
        </a>
        ). However, in the Stbm phosphomutant, Stbm still appeared to cluster
        into puncta, despite the loss of asymmetry (
        <a href="#fig2">
          Figure 2E
        </a>
        ).
      </p>
      <p>
        To quantitate puncta size, we thresholded images using the same
        threshold value in wild-type and mutant regions of the same wings. Fmi
        co-immunolabelling was used to select puncta, as overall junctional
        levels of Fmi do not change in either the phosphomutant or the
        phosphomimetic tissue (see below). This revealed that a similar number
        of puncta were seen in the phosphomutant as in wild-type tissue, while
        many fewer puncta were seen in the phosphomimetic (
        <a href="#fig2">
          Figure 2L
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ). The mean amount of Stbm in puncta was however slightly reduced in
        both phosphomutant and phosphomimetic tissue (
        <a href="#fig2">
          Figure 2M
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig2sdata1">
          Figure 2—source data 1
        </a>
        ). As the phosphomutant forms a similar number of puncta to wild-type,
        this suggests that the coupling between puncta formation and core
        protein asymmetry is lost in Stbm phosphomutants.
      </p>
      <p>
        Other core proteins co-localised with both phosphomutant and
        phosphomimetic Stbm (
        <a href="#fig3">
          Figure 3A–D
        </a>
        ). Overall levels of Fmi and Dsh in junctions were similar to wild-type,
        while Fz levels were slightly decreased in both cases, and Pk levels
        were lower in phosphomutant and higher in phosphomimetic wings (
        <a href="https://elifesciences.org/articles/45107/figures#fig3s1">
          Figure 3—figure supplement 1A–J
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig3sdata1">
          Figure 3—source data 1
        </a>
        ).
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig3-v2.tif/full/,1500/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig3-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Stbm phosphomutants form ‘mixed’ puncta.
          </h6>
          <p>
            (
            <strong>
              A–D
            </strong>
            ) 28 hr APF pupal wings expressing
            <em>
              P[acman]-stbmS[All]A stbm
              <sup>
                6
              </sup>
            </em>
            (
            <strong>
              A,C
            </strong>
            ) or
            <em>
              P[acman]-stbmS[All]E stbm
              <sup>
                6
              </sup>
            </em>
            (
            <strong>
              B,D
            </strong>
            ). Blue immunolabelled tissue in (
            <strong>
              D
            </strong>
            ) is wild-type. Wings immunolabelled for Stbm in green …
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.007">
            https://doi.org/10.7554/eLife.45107.007
          </a>
        </figcaption>
      </figure>

      <h3>
        Stbm phosphomutant puncta contain complexes in both orientations
      </h3>

      <p>
        Core protein complexes within junctional puncta are highly polarised
        compared to other junctional regions (
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ;
        <a href="#bib16">
          Cho et al., 2015
        </a>
        ;
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). This is consistent with puncta containing arrays of core protein
        complexes, all aligned in the same direction (
        <a href="#fig1">
          Figure 1C
        </a>
        , right). As Stbm phosphomutant puncta are no longer associated with
        overall asymmetry we asked whether individual puncta are still
        polarised, or whether phosphomutant puncta have a different
        organisation. To test this, we made adjacent twin clones of EGFP-tagged
        Dgo next to mApple-tagged Dgo. In both wild-type and phosphomutant
        backgrounds, EGFP-Dgo and mApple-Dgo co-localise with other core
        proteins in puncta, as expected (
        <a href="https://elifesciences.org/articles/45107/figures#fig3s1">
          Figure 3—figure supplement 1K and L
        </a>
        ). In a wild-type background, Dgo localised predominantly to distal cell
        ends, as previously reported (
        <a href="#bib18">
          Das et al., 2004
        </a>
        ), so puncta on clone boundaries contained either EGFP-Dgo (green) or
        mApple-Dgo (red) (
        <a href="#fig3">
          Figure 3E,G
        </a>
        ). We could envisage two scenarios for Stbm phosphomutant puncta on
        clone boundaries: if individual puncta were polarised, we would expect
        to see puncta containing either EGFP-Dgo or mApple-Dgo, but Dgo in
        puncta could localise to any cell edge (
        <a href="#fig3">
          Figure 3F
        </a>
        , left). Alternatively if individual puncta were not polarised,
        co-localisation of EGFP-Dgo and mApple-Dgo would be seen (
        <a href="#fig3">
          Figure 3F
        </a>
        , right). Such co-localisation of EGFP-Dgo and mApple-Dgo was indeed
        observed (
        <a href="#fig3">
          Figure 3H
        </a>
        ). This indicates that individual puncta (at least at this optical
        resolution) contain complexes in both orientations (
        <a href="#fig3">
          Figure 3I
        </a>
        ), and that inhibition of Stbm phosphorylation disrupts sorting of
        complexes.
      </p>

      <h3>
        Phosphorylation of Stbm regulates its turnover at junctions
      </h3>

      <p>
        In a wild-type situation, the alignment of core proteins in the same
        orientation within puncta correlates with low protein turnover, as
        measured by Fluorescence Recovery After Photobleaching (FRAP) assays (
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ). As the Stbm phosphomutant forms abnormal puncta with complexes in
        both orientations, we investigated the turnover of phosphomutant Stbm. A
        ‘hub-and-spoke’ FRAP methodology was used, in which the junctions in the
        equivalent of half a cell are bleached (
        <a href="#bib55">
          Warrington et al., 2017
        </a>
        ,
        <a href="#fig4">
          Figure 4A
        </a>
        ). This avoids excessive bleaching of total protein in any single cell,
        while allowing junctions of all orientations to be sampled, regardless
        of whether they are enriched for core proteins or contain puncta.
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig4-v2.tif/full/1500,/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig4-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Stbm phosphomutants have increased stability at junctions.
          </h6>
          <p>
            (
            <strong>
              A
            </strong>
            ) Schematic of hub-and-spoke FRAP. The orange dotted line shows the
            ‘hub-and-spoke’ region that is bleached, which corresponds to three
            cell vertices and the equivalent of three cell junctions. …
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.010">
            https://doi.org/10.7554/eLife.45107.010
          </a>
        </figcaption>
      </figure>

      <p>
        Hub-and-spoke FRAP showed that the stable amount of phosphomutant
        Stbm-EGFP at junctions was increased compared to wild-type Stbm-EGFP,
        while the unstable amount was unchanged (
        <a href="#fig4">
          Figure 4B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4s1">
          Figure 4—figure supplement 1
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4sdata1">
          Figure 4—source data 1
        </a>
        ). This suggests that Stbm phosphorylation promotes turnover of Stbm.
        Consistent with this, the stable amount of phosphomimetic Stbm-EGFP did
        not change compared to wild-type, but the unstable amount increased (
        <a href="#fig4">
          Figure 4B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4s1">
          Figure 4—figure supplement 1
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4sdata1">
          Figure 4—source data 1
        </a>
        ). This suggests that phosphomimetic Stbm accumulates at junctions even
        when not stably incorporated into complexes.
      </p>
      <p>
        Taken together with our previous data that there is less phosphomimetic
        Stbm incorporated into puncta (
        <a href="#fig2">
          Figure 2L,M
        </a>
        ), this supports a model in which phosphorylation promotes Stbm
        turnover, while lack of phosphorylation promotes Stbm clustering in
        complexes (
        <a href="#fig4">
          Figure 4C
        </a>
        ). As both protein turnover and clustering are thought to be required
        for sorting of core proteins to opposite cell ends, these data are
        consistent with the idea that Stbm normally cycles between
        phosphorylated and unphosphorylated states during complex sorting and
        establishment of asymmetry.
      </p>
      <p>
        We also measured the turnover of Stbm
        <em>
          within
        </em>
        puncta (
        <a href="#fig4">
          Figure 4D
        </a>
        ), as these are normally sites of high core protein asymmetry and
        stability, but contain complexes of mixed orientation in Stbm
        phosphomutants. In keeping with the hub-and-spoke FRAP, an increase in
        the stable fraction of phosphomutant Stbm-EGFP (i.e. decreased turnover)
        was seen in puncta, while the stable fraction was decreased in
        phosphomimetic Stbm-EGFP puncta (
        <a href="#fig4">
          Figure 4E
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4sdata1">
          Figure 4—source data 1
        </a>
        , note that as puncta size varies between genotypes we were not able to
        translate stable fractions into stable amounts).
      </p>
      <p>
        As core protein complexes are thought to be sorted via feedback
        interactions, we considered how the altered stability of Stbm
        phosphomutants and phosphomimetics affected the localisation and
        stability of other core proteins. Interestingly, the stable fraction of
        Fz-EGFP and Fmi-EGFP within puncta was decreased in both a Stbm
        phosphomutant and a Stbm phosphomimetic background (
        <a href="#fig4">
          Figure 4F and G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4sdata1">
          Figure 4—source data 1
        </a>
        ). We interpret this to mean that inhibiting Stbm phosphorylation
        promotes excess clustering and stability of Stbm within complexes; but
        the presence of oppositely oriented complexes may promote negative
        feedback interactions, leading to destabilisation of other complex
        components. In phosphomutant Stbm wings, this competition between
        complex stabilisation and destabilisation results in a net increase in
        Stbm stability, but a net decrease in Fmi and Fz stability. In the
        phosphomimetic, the result is a net decrease in stability of all three
        core proteins. In both cases negative feedback between unsorted
        complexes may prevent puncta growing to the same size as wild-type
        puncta (see
        <a href="#fig2">
          Figure 2M
        </a>
        ).
      </p>
      <p>
        A role for Stbm in promoting clustering of complexes of the same
        orientation may be a mechanism for feedback amplification of asymmetry.
        Interestingly, wings expressing one copy of either phosphomutant or
        phosphomimetic Stbm and one copy of wild-type Stbm failed to polarise,
        suggesting that both mutant forms act as dominant negatives (
        <a href="https://elifesciences.org/articles/45107/figures#fig4s2">
          Figure 4—figure supplement 2
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig4sdata1">
          Figure 4—source data 1
        </a>
        , see also
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). Large puncta were observed in the phosphomutant heterozygotes (
        <a href="https://elifesciences.org/articles/45107/figures#fig4s2">
          Figure 4—figure supplement 2B
        </a>
        ), consistent with a model in which excess clustering of phosphomutant
        Stbm leads to recruitment of wild-type Stbm into abnormal, mixed puncta.
      </p>

      <h3>
        In vivo regulation of Stbm by Dco kinase
      </h3>

      <p>
        Experiments in cultured cells have suggested that at least some of the
        phosphorylation of Stbm or Vangl2 could be mediated by the kinase CKIε
        (Dco in flies,
        <a href="#bib23">
          Gao et al., 2011
        </a>
        ;
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). However, this has proven difficult to verify in vivo. Stbm migration
        on SDS-PAGE was not altered in
        <em>
          dco
        </em>
        hypomorphs (
        <a href="https://elifesciences.org/articles/45107/figures#fig5s1">
          Figure 5—figure supplement 1A
        </a>
        , see also
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ). However, we did see a subtle increase in Stbm migration after
        expression of dominant-negative Dco in pupal wings (
        <a href="https://elifesciences.org/articles/45107/figures#fig5s1">
          Figure 5—figure supplement 1B
        </a>
        ).
      </p>
      <p>
        To confirm a role for Dco in regulating Stbm phosphorylation, we
        examined the turnover of Stbm by FRAP when either dominant-negative or
        wild-type Dco were overexpressed. Overexpression of dominant-negative
        Dco led to an increase in the stable amount of Stbm-EGFP at junctions,
        phenocopying the results from Stbm phosphomutants (
        <a href="#fig5">
          Figure 5A
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5s2">
          Figure 5—figure supplement 2A–C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5sdata1">
          Figure 5—source data 1
        </a>
        ). Expression of dominant-negative Dco had no further effect on the
        stable amount of phosphomutant Stbm (
        <a href="#fig5">
          Figure 5A
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5s2">
          Figure 5—figure supplement 2A–C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5sdata1">
          Figure 5—source data 1
        </a>
        ). Conversely, overexpression of wild-type Dco, which would be expected
        to promote excess phosphorylation, caused an increase in the unstable
        amount of Stbm, similar to the Stbm phosphomimetic (
        <a href="#fig5">
          Figure 5B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5s2">
          Figure 5—figure supplement 2D–F
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5sdata1">
          Figure 5—source data 1
        </a>
        ).
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig5-v2.tif/full/1500,/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig5-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Regulation of Stbm phosphorylation and turnover by Dco.
          </h6>
          <p>
            (
            <strong>
              A–C
            </strong>
            ) Stable and unstable amounts of EGFP-tagged protein, in 28 hr APF
            pupal wings after hub-and-spoke FRAP. Flies were (
            <strong>
              A
            </strong>
            )
            <em>
              P[acman]-stbm-EGFP stbm
              <sup>
                6
              </sup>
              /+
            </em>
            (n = 11),
            <em>
              P[acman]-stbm-EGFP stbm
              <sup>
                6
              </sup>
              /en-GAL4; …
            </em>
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.014">
            https://doi.org/10.7554/eLife.45107.014
          </a>
        </figcaption>
      </figure>

      <p>
        As Dco is known to act on Dsh as well as Stbm, we then asked whether the
        effect of Dco on Stbm turnover was independent of any effect on Dsh.
        Overall levels of Stbm are decreased in a
        <em>
          dsh
          <sup>
            1
          </sup>
        </em>
        mutant background (a planar polarity-specific mutation), leading to a
        decrease in the absolute amount of stable Stbm (
        <a href="#fig6">
          Figure 6B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s1">
          Figure 6—figure supplement 1C–E
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). However, expression of dominant-negative Dco in a
        <em>
          dsh
        </em>
        mutant still caused an increase in the stable amount of Stbm (
        <a href="#fig5">
          Figure 5C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5s2">
          Figure 5—figure supplement 2G–I
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig5sdata1">
          Figure 5—source data 1
        </a>
        ). Therefore, we conclude that Dco affects Stbm turnover independently
        of Dsh and supports a model in which Dco regulates Stbm turnover by
        direct phosphorylation of Stbm.
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig6-v2.tif/full/,1500/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig6-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Pk reduces Stbm phosphorylation and promotes Stbm stability.
          </h6>
          <p>
            (
            <strong>
              A,C,E
            </strong>
            ) Western blots probed with Stbm antibody, of pupal wing extracts. (
            <strong>
              A
            </strong>
            ) Wings from wild-type,
            <em>
              fz
              <sup>
                P21
              </sup>
            </em>
            ,
            <em>
              pk
              <sup>
                pk-sple13
              </sup>
            </em>
            ,
            <em>
              dsh
              <sup>
                1
              </sup>
            </em>
            or
            <em>
              P[acman]-stbmS[All]A stbm
              <sup>
                6
              </sup>
            </em>
            flies at 28 hr APF, two biological …
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.018">
            https://doi.org/10.7554/eLife.45107.018
          </a>
        </figcaption>
      </figure>

      <h3>
        Stbm phosphorylation and turnover are negatively regulated by Pk
      </h3>

      <p>
        If Stbm phosphorylation normally controls Stbm turnover and this is
        important for core protein asymmetry, we considered the possibility that
        Stbm phosphorylation might be regulated by other core proteins.
        Interestingly, we saw a decrease in Stbm migration on SDS-PAGE in a
        <em>
          pk
        </em>
        mutant background (
        <a href="#fig6">
          Figure 6A
        </a>
        ). This decrease in migration was lost after phosphatase treatment (
        <a href="https://elifesciences.org/articles/45107/figures#fig6s2">
          Figure 6—figure supplement 2A
        </a>
        ), suggesting that Pk negatively regulates Stbm phosphorylation.
        Consistent with this, loss of Pk did not alter migration of either the
        Stbm phosphomutant or the phosphomimetic on SDS-PAGE (
        <a href="https://elifesciences.org/articles/45107/figures#fig6s2">
          Figure 6—figure supplement 2B
        </a>
        ).
      </p>
      <p>
        Overexpression of Pk causes excess clustering of core proteins into
        large junctional puncta (
        <a href="#bib52">
          Tree et al., 2002
        </a>
        ;
        <a href="#bib6">
          Bastock et al., 2003
        </a>
        ), which contain complexes in both orientations (
        <a href="https://elifesciences.org/articles/45107/figures#fig6s1">
          Figure 6—figure supplement 1A B
        </a>
        ), as seen for Stbm phosphomutant puncta. Interestingly, this also led
        to increased migration of Stbm on SDS-PAGE, suggesting decreased
        phosphorylation (
        <a href="#fig6">
          Figure 6C
        </a>
        ). Thus, Pk overexpression mimics Stbm phosphomutant phenotypes, while
        loss of Pk has the opposite effect.
      </p>

      <p>
        Previous studies have suggested that Fz or Dsh might promote Stbm
        phosphorylation (
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). This would support a model in which Fz or Dsh recruit a kinase, thus
        phosphorylating and destabilising Stbm in complexes in the opposite
        orientation. However, we did not see any change in the migration of
        endogenous Stbm on SDS-PAGE in
        <em>
          fz
        </em>
        or
        <em>
          dsh
        </em>
        mutants, with Stbm still migrating more slowly than phosphomutant Stbm (
        <a href="#fig6">
          Figure 6A
        </a>
        ). This differs from the work of
        <a href="#bib29">
          Kelly et al. (2016)
        </a>
        , who observed increased mobility of FLAG-tagged Stbm in
        <em>
          fz
        </em>
        mutants. We do not know why our results differ, but it is possible that
        tagging Stbm at the C-terminus with FLAG affects its function. Further
        support for our data comes from the observation that the
        hyperphosphorylation seen in
        <em>
          pk
        </em>
        mutants is retained in
        <em>
          pk; fz
        </em>
        or
        <em>
          dsh; pk
        </em>
        mutants, suggesting that neither Fz nor Dsh is needed for this
        hyperphosphorylation (
        <a href="#fig6">
          Figure 6E
        </a>
        ). However, loss of Fz or Dsh did lead to a decrease in the stable
        amounts of Stbm at junctions (
        <a href="#fig6">
          Figure 6B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s1">
          Figure 6—figure supplement 1C–E
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). This could be because Fz and Dsh normally promote Stbm recruitment
        across cell junctions, which would be consistent with the previously
        reported stabilisation of Fz across cell junctions by Stbm and Pk (
        <a href="#bib55">
          Warrington et al., 2017
        </a>
        ).
      </p>

      <h3>
        Regulation of Stbm turnover and clustering by Pk
      </h3>

      <p>
        Hyperphosphorylation of Stbm in a
        <em>
          pk
        </em>
        mutant was accompanied by increased Stbm turnover (
        <a href="#fig6">
          Figure 6B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s1">
          Figure 6—figure supplement 1C–E
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ), suggesting that Pk may stabilise Stbm by inhibiting its
        phosphorylation. In contrast, Pk overexpression leads to decreased Stbm
        turnover (
        <a href="#fig6">
          Figure 6D
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s1">
          Figure 6—figure supplement 1F–H
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ).
      </p>
      <p>
        How might Pk regulate Stbm phosphorylation and turnover? One possibility
        is that the role of Pk is simply to promote complex sorting, which has
        been shown to occur by Pk destabilising Fz within the same cell, acting
        via Dsh (
        <a href="#bib55">
          Warrington et al., 2017
        </a>
        ). It is possible that when complexes sort out into arrays of the same
        orientation, clustering of Stbm leads to reduced accessibility to the
        kinase and thus reduced phosphorylation. We do not favour this
        ‘indirect’ model, as hyperphosphorylation of Stbm is not seen in all
        situations where complexes are thought to be unsorted, for example in
        <em>
          fz
        </em>
        or
        <em>
          dsh
        </em>
        mutants (
        <a href="#fig6">
          Figure 6A
        </a>
        ).
      </p>
      <p>
        An alternative model is that Pk directly regulates Stbm phosphorylation,
        perhaps by regulating its clustering (see Discussion). To investigate
        this, FRAP experiments were carried out on clone boundaries, in which Pk
        activity was present only in the same cell as Stbm, or only in
        neighbouring cells (
        <a href="#fig6">
          Figure 6F
        </a>
        ). In an otherwise wild-type background, Stbm-EGFP strongly accumulates
        on boundaries next to
        <em>
          stbm
        </em>
        mutant cells, where it is highly stable (
        <a href="#fig6">
          Figure 6G
        </a>
        , column 1,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s3">
          Figure 6—figure supplement 3A and E–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). Stbm-EGFP still accumulates on such boundaries in a
        <em>
          pk
        </em>
        mutant background, but its stability is significantly decreased (
        <a href="#fig6">
          Figure 6G
        </a>
        , column 4,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s3">
          Figure 6—figure supplement 3D–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). If Pk is present only in Stbm-EGFP expressing cells, the phenotype
        resembles that of the ‘wild-type’ situation, and Stbm is highly stable (
        <a href="#fig6">
          Figure 6G
        </a>
        , column 2,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s3">
          Figure 6—figure supplement 3B and E–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). In contrast, if Pk is absent only in Stbm-EGFP expressing cells,
        Stbm-EGFP is unstable (
        <a href="#fig6">
          Figure 6G
        </a>
        , column 3,
        <a href="https://elifesciences.org/articles/45107/figures#fig6s3">
          Figure 6—figure supplement 3C and E–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). This indicates that Pk acts in the same cell to stabilise Stbm and
        supports a direct role for Pk in regulating Stbm phosphorylation and
        turnover.
      </p>
      <p>
        If a major role of Pk was to regulate Stbm phosphorylation, and this
        phosphorylation regulates Stbm clustering into puncta, we would also
        expect Stbm phosphomutant clustering to be largely insensitive to loss
        of Pk. In wild-type wings, loss of Pk causes a reduction in puncta size
        (
        <a href="#fig6">
          Figure 6H and K
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ,
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ). As expected, loss of Pk had less effect on either Stbm phosphomutant
        or Stbm phosphomimetic puncta (
        <a href="#fig6">
          Figure 6I–K
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig6sdata1">
          Figure 6—source data 1
        </a>
        ). This again supports the conclusion that Pk directly regulates Stbm
        clustering by modulating the Dco-dependent phosphorylation of Stbm. In
        contrast, loss of Dsh caused a reduction in phosphomutant and
        phosphomimetic puncta size, as is also seen in the presence of wild-type
        Stbm (
        <a href="https://elifesciences.org/articles/45107/figures#fig6s4">
          Figure 6—figure supplement 4A and B
        </a>
        ,
        <a href="#bib46">
          Strutt et al., 2011
        </a>
        ). This is consistent with a model in which Fz and Dsh regulate Stbm
        clustering indirectly by promoting intercellular complex formation.
      </p>

      <h3>
        Regulation of Dsh junctional localisation by Dco kinase in vivo is
        independent of Stbm
      </h3>

      <p>
        Having established that Dco regulates Stbm phosphorylation, and this
        controls the turnover and clustering of Stbm, we next questioned whether
        these mechanisms are sufficient to explain all the effects of Dco on the
        core proteins. In particular, does Dco also directly regulate Dsh in any
        way, or does Dco only regulate Stbm, which then leads to secondary
        effects on Dsh phosphorylation?
      </p>
      <p>
        A number of lines of evidence suggest a direct effect of Dco on Dsh.
        Firstly, we analysed the effect of expressing dominant-negative Dco on
        EGFP-Dsh turnover. We saw a reduction in overall junctional levels of
        EGFP-Dsh, leading to a proportionate decrease in the stable amount of
        Dsh (
        <a href="#fig7">
          Figure 7A
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1A–C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). Junctional levels of EGFP-Dsh were also decreased in the absence of
        Stbm (compare
        <a href="#fig7">
          Figure 7A and B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1E–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ), but were further decreased when dominant-negative Dco was expressed (
        <a href="#fig7">
          Figure 7B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1E–G
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). This argues for a role for Dco in regulating Dsh levels at cell
        junctions independently of Stbm. Furthermore, neither Dsh levels nor Dsh
        phosphorylation were altered in Stbm phosphomutants (
        <a href="#fig7">
          Figure 7C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig3s1">
          Figure 3—figure supplement 1E,F,I and J
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig3sdata1">
          Figure 3—source data 1
        </a>
        ), whereas a decrease in Dsh phosphorylation was seen in
        <em>
          dco
        </em>
        hypomorphs (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s2">
          Figure 7—figure supplement 2A and B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ,
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ).
      </p>

      <figure>
        <a
          href="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig7-v2.tif/full/,1500/0/default.jpg"
        >
          <img
            src="https://iiif.elifesciences.org/lax:45107%2Felife-45107-fig7-v2.tif/full/617,/0/default.jpg"
          />
        </a>
        <figcaption>
          <h6>
            Phosphorylation of Dsh by Dco promotes core protein asymmetry.
          </h6>
          <p>
            (
            <strong>
              A,B
            </strong>
            ) Stable and unstable amounts of EGFP-Dsh in 28 hr APF pupal wings
            after hub-and-spoke FRAP. (
            <strong>
              A
            </strong>
            ) Flies were
            <em>
              dsh
              <sup>
                V26
              </sup>
              /+; P[acman]-EGFP-dsh/+
            </em>
            (n = 14), and
            <em>
              dsh
              <sup>
                V26
              </sup>
              /+; P[acman]-EGFP-dsh/en-GAL4; …
            </em>
          </p>
          <a href="https://doi.org/10.7554/eLife.45107.024">
            https://doi.org/10.7554/eLife.45107.024
          </a>
        </figcaption>
      </figure>

      <p>
        We then examined whether regulation of Dsh by Dco is important in
        establishing asymmetry. As discussed above, expression of dominant
        negative Dco caused a decrease in total levels of Dsh at junctions (
        <a href="#fig7">
          Figure 7A
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). However, a decrease in Dsh levels alone is not sufficient to affect
        asymmetry: halving Dsh dosage halves Dsh levels at junctions (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ), with little effect on the stable fraction (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s1">
          Figure 7—figure supplement 1D
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ) or asymmetry (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ).
      </p>
      <p>
        However, two lines of evidence suggest that Dco-mediated phosphorylation
        of Dsh is functionally important for asymmetry. Firstly, overexpression
        of Dco caused strong trichome swirling in the adult wing (
        <a href="#fig7">
          Figure 7D
        </a>
        ,
        <a href="#bib17">
          Cong et al., 2004
        </a>
        ;
        <a href="#bib30">
          Klein et al., 2006
        </a>
        ;
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ), and this was accompanied by excess Dsh phosphorylation (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s2">
          Figure 7—figure supplement 2C and D
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). Notably, these trichome polarity defects were suppressed by halving
        <em>
          dsh
        </em>
        gene dosage, but not
        <em>
          stbm
        </em>
        or
        <em>
          fz
        </em>
        gene dosage (
        <a href="#fig7">
          Figure 7E–H
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). This genetic interaction supports a direct role for Dco in regulating
        Dsh phosphorylation and core protein asymmetry.
      </p>
      <p>
        Secondly, we analysed
        <em>
          dsh
        </em>
        mutant flies carrying a
        <em>
          dsh
        </em>
        genomic rescue construct in which eight putative Dco phosphorylation
        sites were mutated to alanine (
        <em>
          dshST8-GFP
        </em>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7s3">
          Figure 7—figure supplement 3A
        </a>
        ,
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ). These flies exhibit only mild defects in trichome polarity (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ), but core protein asymmetry was not previously examined. As expected,
        core protein asymmetry was normal in
        <em>
          dsh
        </em>
        mutant flies carrying a wild-type Dsh rescue construct (
        <a href="#fig7">
          Figure 7I and K
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). However, core protein asymmetry was much reduced in
        <em>
          dshST8-GFP
        </em>
        flies (
        <a href="#fig7">
          Figure 7J and K
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ), despite levels at junctions being similar to levels of wild-type Dsh
        (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s3">
          Figure 7—figure supplement 3B
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ). Interestingly, DshST8-GFP has a small but significantly increased
        rate of turnover at cell junctions, compared to wild-type Dsh-GFP (
        <a href="https://elifesciences.org/articles/45107/figures#fig7s3">
          Figure 7—figure supplement 3C
        </a>
        ,
        <a href="https://elifesciences.org/articles/45107/figures#fig7sdata1">
          Figure 7—source data 1
        </a>
        ).
      </p>
      <p>
        Taken together, these results support a model in which Dco-mediated
        phosphorylation of Dsh regulates its recruitment into junctional
        complexes and that this is essential for core protein asymmetry.
      </p>

      <h2>
        Discussion
      </h2>

      <p>
        In this paper, we describe a dual role for CKIε/Dco kinase in regulating
        planar polarity in the fly pupal wing. In the first case, Dco promotes
        phosphorylation of Stbm. Stbm phosphorylation acts as a switch, changing
        Stbm from a stable immobile form that can enter junctional complexes, to
        an unstable mobile form that can redistribute within cells (
        <a href="#fig4">
          Figure 4C
        </a>
        ). Inhibiting Stbm phosphorylation causes an increase in Stbm stability
        at junctions that prevents sorting of complexes: thus complexes are
        ‘locked’ in an unsorted state. In contrast, hyperphosphorylation of Stbm
        destabilises Stbm, allowing it to leave junctions, hence permitting
        complex sorting. A second role for Dco is to mediate Dsh
        phosphorylation, which increases Dsh localisation at junctions.
        Significantly, the effects of Dco on Dsh are independent of Stbm and
        vice versa.
      </p>
      <p>
        In our ‘cloud model’ (
        <a href="#fig1">
          Figure 1C
        </a>
        ,
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ), we envisage that multiple binding interactions drive a phase
        transition from a loosely packed, disordered association of core
        proteins in non-puncta, towards a highly cross-linked array of complexes
        within puncta that are all aligned in the same orientation. Stbm is
        well-placed to be a key component driving such a clustering mechanism,
        as not only can it multimerise with itself (
        <a href="#bib27">
          Jenny et al., 2003
        </a>
        ), but it also has a high stoichiometry within junctions (
        <a href="#fig1">
          Figure 1C
        </a>
        ,
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). Also consistent with a role for Stbm in complex clustering is the
        observation that Stbm phosphorylation site mutants act as dominant
        negatives, recruiting wild-type Stbm into non-polarised puncta.
        Phosphorylation may inhibit a clustering mechanism, due to an increase
        in negative charge (reviewed in
        <a href="#bib57">
          Wu, 2013
        </a>
        ;
        <a href="#bib8">
          Bergeron-Sandoval et al., 2016
        </a>
        ;
        <a href="#bib9">
          Boeynaems et al., 2018
        </a>
        ).
      </p>
      <p>
        Interestingly, excess clustering of unphosphorylated Stbm in unsorted
        complexes is also expected to lead to destabilising feedback
        interactions with the other core components. When Stbm is
        unphosphorylated, the increase in Stbm stability is sufficient for Stbm
        to ‘win’ over Fmi and Fz. Thus, there is an overall increase in Stbm
        stability in phosphomutant Stbm puncta, that is accompanied by decreased
        stability of Fmi and Fz (
        <a href="#fig4">
          Figure 4E–G
        </a>
        ).
      </p>
      <p>
        Pk both promotes Stbm stability and reduces its phosphorylation. A role
        for Pk in increasing Stbm stability is not surprising, as overexpression
        of Pk is known to cause excess clustering of the core proteins (
        <a href="#bib6">
          Bastock et al., 2003
        </a>
        ;
        <a href="#bib52">
          Tree et al., 2002
        </a>
        ). We can envisage a number of mechanisms by which Pk could affect Stbm
        phosphorylation.
        <a href="#bib55">
          Warrington et al. (2017)
        </a>
        provided evidence that Pk has two roles: firstly, it acts via Dsh to
        destabilise Fz in the same cell (
        <a href="#fig8">
          Figure 8A
        </a>
        ); secondly, it acts via Stbm to stabilise Fz in adjacent cells (
        <a href="#fig8">
          Figure 8B
        </a>
        ). In the first case, Pk would promote sorting of complexes, and one
        possibility is that Stbm is inaccessible to the kinase in sorted
        complexes, and thus Pk is indirectly reducing Stbm phosphorylation by
        promoting sorting. Arguing against this, loss of
        <em>
          fz
        </em>
        or
        <em>
          dsh
        </em>
        also abolishes core protein asymmetry, but no hyperphosphorylation is
        seen. Our boundary FRAP experiments instead support Pk acting directly
        in the same cell to stabilise Stbm. We therefore propose a mechanism
        whereby direct binding of Pk to Stbm protects Stbm from phosphorylation.
      </p>
      <p>
        Interestingly, Stbm has a significantly higher stoichiometry within
        junctions than Pk (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). One possibility is that Stbm forms multimers, and that association of
        Pk with these multimers causes a conformational change that reduces
        accessibility to kinase-binding sites. Alternatively, Pk might recruit a
        phosphatase (albeit no candidates for such a phosphatase are known). The
        reduced negative charge might then allow Stbm to form higher order
        structures, which promotes clustering of the entire core protein complex
        into puncta (
        <a href="https://elifesciences.org/articles/45107/figures#fig6s4">
          Figure 6—figure supplement 4C
        </a>
        ).
      </p>
      <p>
        Puncta formation in both wild-type and phosphomutants is also dependent
        on Dsh. Dsh is another a good candidate for promoting clustering as it
        too can multimerise (
        <a href="#bib41">
          Schwarz-Romond et al., 2007
        </a>
        ;
        <a href="#bib22">
          Gammons et al., 2016
        </a>
        ), and thus puncta formation may be dependent on clustering on both
        sides of the complex. Moreover, direct interactions between Stbm and Dsh
        (
        <a href="#bib35">
          Park and Moon, 2002
        </a>
        ;
        <a href="#bib6">
          Bastock et al., 2003
        </a>
        ) may promote clustering of unsorted complexes in the absence of
        phosphorylation.
      </p>
      <p>
        Feedback models for core protein asymmetry suggest that particular
        components of the core pathway signal to other components to either
        stabilise or destabilise them (
        <a href="#bib2">
          Amonlirdviman et al., 2005
        </a>
        ;
        <a href="#bib11">
          Burak and Shraiman, 2009
        </a>
        ;
        <a href="#bib31">
          Le Garrec et al., 2006
        </a>
        ;
        <a href="#bib40">
          Schamberg et al., 2010
        </a>
        ). An attractive model would be that Fz or Dsh recruits a kinase which
        phosphorylates Stbm and destabilises complexes of the opposite
        orientation (
        <a href="#bib29">
          Kelly et al., 2016
        </a>
        ;
        <a href="#bib58">
          Yang et al., 2017
        </a>
        ). Consistent with this, a proportion of Dco localises to apicolateral
        junctions in pupal wings (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ). However, we do not see any change in Stbm phosphorylation in
        <em>
          fz
        </em>
        or
        <em>
          dsh
        </em>
        mutants, nor are Fz and Dsh required for the hyperphosphorylation of
        Stbm seen in
        <em>
          pk
        </em>
        mutants. Therefore, we conclude that Stbm phosphorylation is more likely
        to be constitutive. Such constitutive phosphorylation would be
        sufficient to keep Stbm mobile and allow complex sorting (
        <a href="#fig8">
          Figure 8C
        </a>
        ); and Pk would then counterbalance this and promote complex stability (
        <a href="#fig8">
          Figure 8B
        </a>
        ). The balance between Stbm phosphorylation/complex mobility and Pk
        binding (leading to reduced Stbm phosphorylation) would resolve over
        time towards a more stable state as complexes segregate to opposite cell
        edges.
      </p>
      <p>
        We note that in normal development, Stbm downregulates Pk levels (
        <a href="#bib48">
          Strutt et al., 2013b
        </a>
        ). This suggests Pk levels are finely tuned, in order to prevent
        unrestrained clustering (as seen when Pk is overexpressed).
      </p>
      <p>
        We also provide evidence that Dco regulates Dsh phosphorylation and
        junctional levels independently of Stbm. Our findings are consistent
        with previous observations that Dsh phosphorylation correlates with its
        recruitment by Fz into junctional complexes (
        <a href="#bib5">
          Axelrod, 2001
        </a>
        ;
        <a href="#bib43">
          Shimada et al., 2001
        </a>
        ). The mechanism by which Dsh phosphorylation acts in planar polarity
        remains to be elucidated, but our data show that
        <em>
          dco
        </em>
        overexpression phenotypes are suppressed by reduced
        <em>
          dsh
        </em>
        gene dosage, and that Dsh phosphomutants have reduced core protein
        asymmetry in pupal wings. Furthermore, a small but significant decrease
        in Dsh stability at junctions is observed in Dsh phosphomutants.
        Overall, our data are consistent with a model in which phosphorylation
        of Dsh promotes its stable association at junctions (
        <a href="#fig8">
          Figure 8B
        </a>
        ).
      </p>
      <p>
        In summary, we propose that Dco regulates the asymmetric localisation of
        the core proteins by reciprocal actions on Stbm and Dsh. Dco regulates
        Stbm phosphorylation and turnover and causes it to leave junctions,
        while phosphorylation of Dsh by Dco promotes its junctional association.
      </p>

      <h2>
        Materials and methods
      </h2>

      <figure>
        <table>
          <thead>
            <tr>
              <th>
                Reagent type (species) or resource
              </th>
              <th>
                Designation
              </th>
              <th>
                Source or reference
              </th>
              <th>
                Identifiers
              </th>
              <th>
                Additional information
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
              <td>
                <a href="#bib56">
                  Wolff and Rubin, 1998
                </a>
                , PMID 9463361
              </td>
              <td>
                BDSC:6918; FLYB:FBal0062423; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_6918">
                  BDSC_6918
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  Vang
                  <sup>
                    stbm-6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>
              </td>
              <td>
                <a href="#bib25">
                  Gubb et al., 1999
                </a>
                , PMID:10485852
              </td>
              <td>
                BDSC:41790; FLYB:FBal0060943; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_41790">
                  BDSC_41790
                </a>
              </td>
              <td>
                FlyBase symbol:

                <em>
                  pk
                  <sup>
                    pk-sple-13
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  dsh
                  <sup>
                    V26
                  </sup>
                </em>
              </td>
              <td>
                <a href="#bib37">
                  Perrimon and Mahowald, 1987
                </a>
                , PMID:3803719
              </td>
              <td>
                BDSC:6331; FLYB:FBal0003140; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_6331">
                  BDSC_6331
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  dsh
                  <sup>
                    3
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  dsh
                  <sup>
                    1
                  </sup>
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:5298; FLYB FBal0003138; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_5298">
                  BDSC_5298
                </a>
              </td>
              <td></td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  dgo
                  <sup>
                    380
                  </sup>
                </em>
              </td>
              <td>
                <a href="#bib21">
                  Feiguin et al., 2001
                </a>
                , PMID:11703927
              </td>
              <td>
                BDSC:41786; FLYB:FBal0141190; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_41786">
                  BDSC_41786
                </a>
              </td>
              <td></td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  fmi-EGFP
                </em>
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                Knock-in of EGFP to the C-terminus of
                <em>
                  fmi
                </em>
                in the endogenous locus
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  fz-EGFP
                </em>
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                Knock-in of EGFP to the C-terminus of
                <em>
                  fz
                </em>
                in the endogenous locus
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-dsh
                </em>
                attP40 (2L) 25C6
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                <em>
                  P[acman
                </em>
                ] transgene with EGFP recombineered at the N-terminus of
                <em>
                  dsh
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-dgo attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with EGFP recombineered at the N-terminus of
                <em>
                  dgo
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbm attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbm-EGFP attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with EGFP recombineered at the C-terminus of
                <em>
                  stbm
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbm-EGFP VK1
                </em>
                (2R) 59D3
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with EGFP recombineered at the C-terminus of
                <em>
                  stbm
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-stbm attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with EGFP recombineered at the N-terminus of
                <em>
                  stbm
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-mApple-stbm attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with mApple recombineered at the N-terminus of
                <em>
                  stbm
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-mApple-dgo attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with mApple recombineered at the N-terminus of
                <em>
                  dgo
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-stbmS[5,120,122]A attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  P[acman]
                </em>
                transgene with EGFP at the N-terminus of
                <em>
                  stbm
                </em>
                , and with phosphorylation site mutations S[5,120,122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-stbmS[5,120,122]E attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with EGFP at the N-terminus of stbm, and with
                phosphorylation site mutations S[5,120,122]E
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-EGFP-stbmS[All]A attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with EGFP at the N-terminus of stbm, and with
                phosphorylation site mutations S[5-17, 113-122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[All]A attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations S[5-17,
                113-122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[All]A VK31
                </em>
                (3L) 62E1
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations S[5-17,
                113-122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[All]E attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations S[5-17,
                113-122]E
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[5-17]A attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations S[5-17]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[113-122]A attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations
                S[113-122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[5-17]E attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations S[5-17]E
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[113-122]E attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with phosphorylation site mutations
                S[113-122]E
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[All]A-EGFP attP40
                </em>
                (2L) 25C6
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with EGFP at the C-terminus of stbm, and with
                phosphorylation site mutations S[5-17, 113-122]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  P[acman]-stbmS[All]E-EGFP VK31
                </em>
                (3L) 62E1
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                P[acman] transgene with EGFP at the C-terminus of stbm, and with
                phosphorylation site mutations S[5-17, 113-122]E
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  attB-UAS-pk VK20
                </em>
                (3R) 99F8
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  pk
                </em>
                gene under control of the
                <em>
                  UAS
                </em>
                promoter
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  attB-UAS-dgo VK20
                </em>
                (3R) 99F8
              </td>
              <td>
                This paper
              </td>
              <td></td>
              <td>
                <em>
                  dgo
                </em>
                gene under control of the
                <em>
                  UAS
                </em>
                promoter
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  CaSpeR-dsh-GFP
                </em>
              </td>
              <td>
                <a href="#bib5">
                  Axelrod, 2001
                </a>
                , PMID:11358862
              </td>
              <td></td>
              <td>
                <em>
                  dsh
                </em>
                genomic rescue construct, with GFP at the C-terminus
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  CaSpeR-dshST8-GFP
                </em>
              </td>
              <td>
                <a href="#bib45">
                  Strutt et al., 2006
                </a>
                , PMID:16824921
              </td>
              <td></td>
              <td>
                <em>
                  dsh
                </em>
                genomic rescue construct with GFP at the C-terminus, and with
                phosphorylation site mutations S[236-247]A
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  UAS-dco
                  <sup>
                    K38R
                  </sup>
                </em>
              </td>
              <td>
                <a href="#bib45">
                  Strutt et al., 2006
                </a>
                , PMID:16824921
              </td>
              <td></td>
              <td>
                Dominant negative
                <em>
                  dco
                </em>
                , under control of the
                <em>
                  UAS
                </em>
                promoter
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  UAS-dco
                </em>
              </td>
              <td>
                <a href="#bib42">
                  Sekine et al., 2008
                </a>
                , PMID:18258753
              </td>
              <td></td>
              <td>
                <em>
                  dco
                </em>
                gene under control of the
                <em>
                  UAS
                </em>
                promoter
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  ptc-GAL4
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:2017; FLYB:FBti0002124; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_2017">
                  BDSC_2017
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{GawB}ptc559.1
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  en-GAL4
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:30564; FLYB:FBti0003572; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_30564">
                  BDSC_30564
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{en2.4-GAL4}e16E
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  Actin-GAL4
                  <sup>
                    25
                  </sup>
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:3953; FLYB:FBti0012293; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_3953">
                  BDSC_3953
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{AyGAL4}25
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  tub-GAL80
                  <sup>
                    ts20
                  </sup>
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:7019; FLYB:FBti0027796; RRID:
                <a href="https://scicrunch.org/resolver/BDSC_7019">
                  BDSC_7019
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{tubP-GAL80ts}
                  <sup>
                    20
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  Ubx-FLP
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:42718; FLYB:FBti0150334: RRID:
                <a href="https://scicrunch.org/resolver/BDSC_42718">
                  BDSC_42718
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{Ubx-FLP}1
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Genetic reagent (
                <em>
                  Drosophila melanogaster
                </em>
                )
              </td>
              <td>
                <em>
                  hs-FLP
                  <sup>
                    G5
                  </sup>
                  attP2
                </em>
              </td>
              <td>
                Bloomington Drosophila Stock Center
              </td>
              <td>
                BDSC:55816; FLYB:FBti0160507: RRID:
                <a href="https://scicrunch.org/resolver/BDSC_55816">
                  BDSC_55816
                </a>
              </td>
              <td>
                FlyBase symbol:
                <em>
                  P{hs-FLP
                  <sup>
                    G5
                  </sup>
                  }attP2
                </em>
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                rabbit polyclonal anti-Stbm
              </td>
              <td>
                <a href="#bib54">
                  Warrington et al., 2013
                </a>
                , PMID:23364328
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2570077">
                  AB_2570077
                </a>
              </td>
              <td>
                1/1000 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                rat polyclonal anti-Stbm
              </td>
              <td>
                <a href="#bib51">
                  Strutt and Strutt, 2008
                </a>
                , PMID:18804371
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2569716">
                  AB_2569716
                </a>
              </td>
              <td>
                1/1000 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                affinity purified rabbit polyclonal anti-Fz
              </td>
              <td>
                <a href="#bib7">
                  Bastock and Strutt, 2007
                </a>
                , PMID:17652348
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2801421">
                  AB_2801421
                </a>
              </td>
              <td>
                1/300 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                affinity purified rat polyclonal anti-Pk
              </td>
              <td>
                <a href="#bib47">
                  Strutt et al., 2013a
                </a>
                , PMID:23487316
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2569720">
                  AB_2569720
                </a>
              </td>
              <td>
                1/25 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                rat polyclonal anti-Dsh
              </td>
              <td>
                <a href="#bib45">
                  Strutt et al., 2006
                </a>
                , PMID:16824921
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2801419">
                  AB_2801419
                </a>
              </td>
              <td>
                1/1000 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                mouse monoclonal anti-Fmi #74
              </td>
              <td>
                <a href="#bib53">
                  Usui et al., 1999
                </a>
                , PMID:10490098
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2619583">
                  AB_2619583
                </a>
              </td>
              <td>
                3 µg/ml (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                affinity purified rabbit polyclonal anti-GFP
              </td>
              <td>
                Abcam
              </td>
              <td>
                Abcam:ab6556; RRID:
                <a href="https://scicrunch.org/resolver/AB_305564">
                  AB_305564
                </a>
              </td>
              <td>
                1/4000 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                mouse monoclonal anti-ß-gal 40-1a
              </td>
              <td>
                DSHB
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2314509">
                  AB_2314509
                </a>
              </td>
              <td>
                1/200 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                rabbit polyclonal anti-ß-gal
              </td>
              <td>
                MP Biomedicals/ Cappel
              </td>
              <td>
                MP Biochemicals:0855976 (Cappel:55976); RRID:
                <a href="https://scicrunch.org/resolver/AB_2334934">
                  AB_2334934
                </a>
              </td>
              <td>
                1/4000 (immunolabelling)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                rabbit polyclonal anti-Stbm
              </td>
              <td>
                <a href="#bib39">
                  Rawls and Wolff, 2003
                </a>
                PMID:12642492
              </td>
              <td></td>
              <td>
                1/20000 (western blotting)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                affinity purified rabbit polyclonal anti-Dsh
              </td>
              <td>
                <a href="#bib45">
                  Strutt et al., 2006
                </a>
                , PMID:16824921
              </td>
              <td>
                RRID:
                <a href="https://scicrunch.org/resolver/AB_2801420">
                  AB_2801420
                </a>
              </td>
              <td>
                1/200 (western blotting)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                mouse monoclonal anti-Actin AC40
              </td>
              <td>
                Sigma-Aldrich
              </td>
              <td>
                Sigma:A4700; RRID:
                <a href="https://scicrunch.org/resolver/AB_476730">
                  AB_476730
                </a>
              </td>
              <td>
                1/5000 (western blotting)
              </td>
            </tr>
            <tr>
              <td>
                Antibody
              </td>
              <td>
                mouse monoclonal anti-Tubulin DM1A
              </td>
              <td>
                Sigma-Aldrich
              </td>
              <td>
                Sigma:T9026; RRID:
                <a href="https://scicrunch.org/resolver/AB_477593">
                  AB_477593
                </a>
              </td>
              <td>
                1/10000 (western blotting)
              </td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                ProgResC14 version 1.7.3
              </td>
              <td>
                Jenoptik
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                NIS Elements AR version 4.60
              </td>
              <td>
                Nikon
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                Image Lab version 4.1
              </td>
              <td>
                BioRad Laboratories
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                ImageJ version 2.0.0-r65/1.51 s
              </td>
              <td>
                <a href="https://fiji.sc/">
                  https://fiji.sc
                </a>
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                Packing Analyzer
              </td>
              <td>
                <a href="#bib1">
                  Aigouy et al., 2010
                </a>
                , PMID:20813263
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                MATLAB_R2014b
              </td>
              <td>
                Mathworks
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                Membrane intensity and Polarity measurement scripts (MATLAB)
              </td>
              <td>
                <a href="#bib49">
                  Strutt et al., 2016
                </a>
                , PMID:27926869
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                Puncta measurement script (MATLAB)
              </td>
              <td>
                <a href="#bib50">
                  Strutt et al., 2019
                </a>
                , PMID: 30661800
              </td>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                Software, algorithm
              </td>
              <td>
                GraphPad Prism version 7.0 c
              </td>
              <td>
                GraphPad Software, Inc.
              </td>
              <td></td>
              <td></td>
            </tr>
          </tbody>
        </table>
      </figure>

      <h3>
        <em>
          Drosophila
        </em>
        genetics
      </h3>

      <p>
        <em>
          Drosophila melanogaster
        </em>
        flies were grown on standard cornmeal/agar/molasses media at 18°C or
        25°C, unless otherwise described.
      </p>
      <p>
        Fly strains are described in FlyBase.
        <em>
          fz
          <sup>
            P21
          </sup>
        </em>
        ,
        <em>
          stbm
          <sup>
            6
          </sup>
        </em>
        ,
        <em>
          pk
          <sup>
            pk-sple13
          </sup>
        </em>
        ,
        <em>
          dsh
          <sup>
            V26
          </sup>
        </em>
        are null alleles, and
        <em>
          dsh
          <sup>
            1
          </sup>
        </em>
        gives a strong planar polarity phenotype, but functions normally in
        Wingless signalling (
        <a href="#bib4">
          Axelrod et al., 1998
        </a>
        ;
        <a href="#bib10">
          Boutros et al., 1998
        </a>
        ).
      </p>
      <p>
        <em>
          P[acman]
        </em>
        constructs (BACPAC resources) were recombineered using standard methods.
        N-terminal fusions of
        <em>
          P[acman]-EGFP-stbm
        </em>
        ,
        <em>
          P[acman]-mApple-stbm
        </em>
        and
        <em>
          P[acman]-mApple-dgo
        </em>
        used plasmid
        <em>
          PL452-N-EGFP
        </em>
        (Addgene) as a source vector, or a modified version
        <em>
          PL452-N-mApple
        </em>
        where mApple replaced EGFP. Gene-specific primers were used to amplify
        EGFP/mApple and the selection cassette, and the resulting fragment was
        then recombineered into the relevant
        <em>
          P[acman]
        </em>
        construct, in frame with the open reading frame. The kanamycin cassette
        was then excised, leaving a single LoxP site between the EGFP/mApple tag
        and the open-reading frame. Phosphomutants were generated using
        recombineering with positive-negative selection into
        <em>
          P[acman]-stbm
        </em>
        (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ),
        <em>
          P[acman]-stbm-EGFP
        </em>
        (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ) or
        <em>
          P[acman]-EGFP-stbm
        </em>
        . These were exact mutations, leaving no foreign sequence. The
        open-reading frame of
        <em>
          dgo
        </em>
        was cloned into
        <em>
          attB-pUAST
        </em>
        using standard methods.
      </p>
      <p>
        <em>
          P[acman]
        </em>
        constructs were integrated into the genome via ΦC31-mediated
        recombination into the
        <em>
          attP40
        </em>
        landing site on 2L, the
        <em>
          VK1
        </em>
        site on 2R or the
        <em>
          VK31
        </em>
        site on 3L.
        <em>
          P[acman]-stbm
        </em>
        lines were recombined or double balanced with
        <em>
          stbm
          <sup>
            6
          </sup>
        </em>
        and
        <em>
          P[acman]-dgo
        </em>
        lines were recombined with
        <em>
          dgo
          <sup>
            380
          </sup>
        </em>
        , or with
        <em>
          stbm
          <sup>
            6
          </sup>
          dgo
          <sup>
            380
          </sup>
          . attB-UAS-pk
        </em>
        (
        <a href="#bib55">
          Warrington et al., 2017
        </a>
        ) and
        <em>
          attB-UAS-dgo
        </em>
        were integrated into the
        <em>
          VK20
        </em>
        landing site on 3L. Transgenics were made by Genetivision.
      </p>
      <p>
        <em>
          fmi-EGFP
        </em>
        and
        <em>
          fz-EGFP
        </em>
        knock-ins and
        <em>
          P[acman]-stbm
        </em>
        ,
        <em>
          P[acman]-stbm-EGFP
        </em>
        ,
        <em>
          P[acman]-EGFP-dsh
        </em>
        and
        <em>
          P[acman]-EGFP-dgo
        </em>
        (all in
        <em>
          attP40
        </em>
        ) are described in
        <a href="#bib49">
          Strutt et al. (2016)
        </a>
        . Other P element transgene insertions were
        <em>
          CaSpeR-dsh-GFP
        </em>
        (
        <a href="#bib5">
          Axelrod, 2001
        </a>
        ),
        <em>
          CaSpeR-dshST8-GFP
        </em>
        (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ),
        <em>
          UAS-dco
          <sup>
            K38R
          </sup>
        </em>
        (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ) and
        <em>
          UAS-dco
        </em>
        (
        <a href="#bib42">
          Sekine et al., 2008
        </a>
        ).
      </p>
      <p>
        Flies were raised at 25°C and dissected or imaged 28 hr after puparium
        formation (APF), unless otherwise indicated. Flies raised at 29°C were
        imaged after 25 hr. To avoid dosage compensation effects, females of
        <em>
          dsh
          <sup>
            V26
          </sup>
          /+; P[acman]-EGFP-dsh/+
        </em>
        were used. Mitotic clones were induced using the FLP/FRT system and
        either
        <em>
          Ubx-FLP
        </em>
        or
        <em>
          hs-FLP
        </em>
        . For expression of
        <em>
          UAS-dco
          <sup>
            K38R
          </sup>
        </em>
        and
        <em>
          UAS-dco
        </em>
        , flies were crossed to
        <em>
          en-GAL4
        </em>
        at 25°C, or
        <em>
          Actin-GAL4, tub-GAL80
          <sup>
            ts
          </sup>
        </em>
        at 29°C. Expression of
        <em>
          attB-UAS-pk
        </em>
        and
        <em>
          attB-UAS-dgo
        </em>
        used
        <em>
          ptc-GAL4
        </em>
        at 25°C or
        <em>
          Actin-GAL4, tub-GAL80
          <sup>
            ts
          </sup>
        </em>
        at 29°C.
      </p>
      <p>
        Full genotypes for each figure are shown in
        <a href="#table1">
          Table 1
        </a>
        .
      </p>

      <figure>
        <figcaption>
          <h6>
            List of full genotypes used in each figure.
          </h6>
          <a href="https://doi.org/10.7554/eLife.45107.030">
            https://doi.org/10.7554/eLife.45107.030
          </a>
        </figcaption>

        <table>
          <thead>
            <tr>
              <th>
                Figure
              </th>
              <th></th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2A
                </a>
              </td>
              <td>
                <em>
                  w; stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2C
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2E, G, I, L, M
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2F, H, I, L, M
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig2">
                  Figure 2J, K
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig3">
                  Figure 3A, C
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig3">
                  Figure 3B, D
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig3">
                  Figure 3G
                </a>
              </td>
              <td>
                <em>
                  w hs-FLP; P[acman]-EGFP-dgo [attP40] FRT40 dgo
                  <sup>
                    380
                  </sup>
                  / P[acman]-mApple-dgo [attP40] FRT40 dgo
                  <sup>
                    380
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig3">
                  Figure 3H
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-dgo [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  dgo
                  <sup>
                    380
                  </sup>
                  / P[acman]-mApple-dgo [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  dgo
                  <sup>
                    380
                  </sup>
                  ; P[acman-stbmS[All]A [VK31] / P[acman]-stbmS[All]A [VK31]
                  hs-FLP
                  <sup>
                    G5
                  </sup>
                  [attP2]
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig4">
                  Figure 4B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]E-EGFP [VK31] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig4">
                  Figure 4E
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]E-EGFP [VK31]
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig4">
                  Figure 4F
                </a>
              </td>
              <td>
                <em>
                  w; fmi-EGFP/+
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT42 stbm
                  <sup>
                    6
                  </sup>
                  fmi-EGFP/P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT42 stbm
                  <sup>
                    6
                  </sup>
                  fmi-EGFP/P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig4">
                  Figure 4G
                </a>
              </td>
              <td>
                <em>
                  w; fz-EGFP/+
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]A [VK31] fz-EGFP / +
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  ; fz-EGFP / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig5">
                  Figure 5A
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[Al]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A [attP40] en-GAL4, stbm
                  <sup>
                    6
                  </sup>
                  ; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig5">
                  Figure 5B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w UAS-dco/w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /en-GAL4
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]E-EGFP [VK31] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig5">
                  Figure 5C
                </a>
              </td>
              <td>
                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6A
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; fz
                  <sup>
                    P21
                  </sup>
                </em>

                <em>
                  w; pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +; fz
                  <sup>
                    P21
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6C
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; Actin-GAL4, tub-GAL80
                  <sup>
                    ts
                  </sup>
                  / +; UAS-pk [VK20] / +w; Actin-GAL4, tub-GAL80
                  <sup>
                    ts
                  </sup>
                  / +; UAS-dgo [VK20] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /Actin-GAL4, tub-GAL80
                  <sup>
                    ts
                  </sup>
                  ; UAS-pk [VK20] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6E
                </a>
              </td>
              <td>
                <em>
                  w; stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w
                </em>

                <em>
                  w; fz
                  <sup>
                    P21
                  </sup>
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                </em>

                <em>
                  w; pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>

                <em>
                  w; pk
                  <sup>
                    pk-sple13
                  </sup>
                  ; fz
                  <sup>
                    P21
                  </sup>
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>

                <em>
                  y w Ubx-FLP; FRT42 stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>

                <em>
                  y w Ubx-FLP; FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>

                <em>
                  y w Ubx-FLP; FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6H, K
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 arm-lacZ / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6I, K
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[All]A [attP40] FRT42 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig6">
                  Figure 6J, K
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[All]E [attP40] FRT42 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]E [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7A
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40]/+
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7B
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] stbm
                  <sup>
                    6
                  </sup>
                  / stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4, stbm
                  <sup>
                    6
                  </sup>
                  ; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7C
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7D, H
                </a>
              </td>
              <td>
                <em>
                  w UAS-dco/w; en-GAL4/+
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7E, H
                </a>
              </td>
              <td>
                <em>
                  w UAS-dco/w; en-GAL4 / +; fz
                  <sup>
                    P21
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7F, H
                </a>
              </td>
              <td>
                <em>
                  w UAS-dco / w; en-GAL4 / stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7G, H
                </a>
              </td>
              <td>
                <em>
                  w UAS-dco / w dsh
                  <sup>
                    V26
                  </sup>
                  ; en-GAL4 / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7I, H
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18; pCaSpeR-dsh-GFP / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a href="#fig7">
                  Figure 7J, K
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18; pCaSpeR-dshST8-GFP / +
                </em>
              </td>
            </tr>
            <tr>
              <td></td>
              <td></td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1A
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-stbmS[5,120,122]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1C
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-stbmS[5,120,122]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1E, I
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-EGFP-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1F, I
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-EGFP-stbmS[5,120,122]A [attP40] FRT40
                  stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-EGFP-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1G, I
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-EGFP-stbmS[5,120,122]E [attP40] FRT40
                  stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-EGFP-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s1"
                >
                  Figure 2—figure supplement 1H, I
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-EGFP-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-EGFP-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2A
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2B
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; P[acman]-stbmS[5-17]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[113-122]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2C, G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[5-17]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2D, H
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[5-17]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2E, G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[113-122]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig2s2"
                >
                  Figure 2—figure supplement 2F, H
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbmS[113-122]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig3s1"
                >
                  Figure 3–figure supplement 1A, C, E, G, I
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  ] / P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig3s1"
                >
                  Figure 3–figure supplement 1B, D, F, H, J
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-stbm [attP40] arm-lacZ FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig3s1"
                >
                  Figure 3—figure supplement 1K
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-dgo [attP40] FRT40 dgo
                  <sup>
                    380
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig3s1"
                >
                  Figure 3—figure supplement 1L
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-EGFP-dgo [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  dgo
                  <sup>
                    380
                  </sup>
                  ; P[acman-stbmS[All]A [VK31]
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s1"
                >
                  Figure 4—figure supplement 1A, C, D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s1"
                >
                  Figure 4—figure supplement 1A, C, E
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s1"
                >
                  Figure 4—figure supplement 1A, C, F
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /
                </em>

                <em>
                  stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]E-EGFP [VK31] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s2"
                >
                  Figure 4—figure supplement 2A-D
                </a>
              </td>
              <td>
                <em>
                  w
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s2"
                >
                  Figure 4—figure supplement 2B, D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig4s2"
                >
                  Figure 4—figure supplement 2C, D
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig5s1"
                >
                  Figure 5—figure supplement 1A
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; FRT82 dco
                  <sup>
                    2
                  </sup>
                  / FRT82 dco
                  <sup>
                    5B2.6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig5s1"
                >
                  Figure 5—figure supplement 1B
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; Actin-GAL4, tub-GAL80
                  <sup>
                    ts
                  </sup>
                  / +; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig5s2"
                >
                  Figure 5—figure supplement 2A-C
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /+w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[Al]A-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-stbmS[All]A [attP40] en-GAL4, stbm
                  <sup>
                    6
                  </sup>
                  ; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig5s2"
                >
                  Figure 5—figure supplement 2D-F
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w UAS-dco / w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / stbm
                  <sup>
                    6
                  </sup>
                  ; P[acman]-stbmS[All]E-EGFP [VK31] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig5s2"
                >
                  Figure 5—figure supplement 2G-I
                </a>
              </td>
              <td>
                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s2"
                >
                  Figure 6—figure supplement 2A
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s2"
                >
                  Figure 6—figure supplement 2B
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]E [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbmS[All]A [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s1"
                >
                  Figure 6—figure supplement 1A
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman]-EGFP-stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-mApple-stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s1"
                >
                  Figure 6—figure supplement 1B
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; P[acman[-EGFP-Stbm [attP40] FRT40 ptc-GAL4 stbm
                  <sup>
                    6
                  </sup>
                  / P[acman]-mApple-Stbm [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  ; UAS-Pk [VK20] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s1"
                >
                  Figure 6—figure supplement 1C-E
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /+
                </em>

                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /+; fz
                  <sup>
                    P21
                  </sup>
                </em>

                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                </em>

                <em>
                  w dsh
                  <sup>
                    1
                  </sup>
                  ; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s1"
                >
                  Figure 6—figure supplement 1F-H
                </a>
              </td>
              <td>
                <em>
                  w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  / +w; P[acman]-stbm-EGFP [attP40] FRT40 stbm
                  <sup>
                    6
                  </sup>
                  /Actin-GAL4, tub-GAL80
                  <sup>
                    ts
                  </sup>
                  ; UAS-pk [VK20] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s3"
                >
                  Figure 6—figure supplement 3A, E-G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s3"
                >
                  Figure 6—figure supplement 3B, E-G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s3"
                >
                  Figure 6—figure supplement 3C, E-G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1] / FRT42 stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s3"
                >
                  Figure 6—figure supplement 3D, E-G
                </a>
              </td>
              <td>
                <em>
                  y w Ubx-FLP; FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  P[acman]-stbm-EGFP [VK1]/ FRT42 pk
                  <sup>
                    pk-sple13
                  </sup>
                  stbm
                  <sup>
                    6
                  </sup>
                  , Ubi-mRFP-nls
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s4"
                >
                  Figure 6—figure supplement 4A
                </a>
              </td>
              <td>
                <em>
                  w dsh
                  <sup>
                    V26
                  </sup>
                  FRT19A/y w Ubx-FLP FRT19A; P(acman)-StbmS(All)A [attP40] FRT40
                  stbm[6]
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig6s4"
                >
                  Figure 6—figure supplement 4B
                </a>
              </td>
              <td>
                <em>
                  w dsh
                  <sup>
                    V26
                  </sup>
                  FRT19A / y w Ubx-FLP FRT19A; P(acman)-StbmS(All)E [attP40]
                  FRT40 stbm[6]
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig7s1"
                >
                  Figure 7—figure supplement 1A-D
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] / +
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] / en-GAL4; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18; P[acman]-EGFP-dsh [attP40] / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig7s1"
                >
                  Figure 7—figure supplement 1E-G
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] stbm
                  <sup>
                    6
                  </sup>
                  / stbm
                  <sup>
                    6
                  </sup>
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18 / w; P[acman]-EGFP-dsh [attP40] stbm
                  <sup>
                    6
                  </sup>
                  / en-GAL4, stbm
                  <sup>
                    6
                  </sup>
                  ; UAS-dco
                  <sup>
                    K38R
                  </sup>
                  / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig7s2"
                >
                  Figure 7—figure supplement 2A, B
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  w; FRT82 dco
                  <sup>
                    2
                  </sup>
                  / FRT82 dco
                  <sup>
                    5B2.6
                  </sup>
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig7s2"
                >
                  Figure 7—figure supplement 2C, D
                </a>
              </td>
              <td>
                <em>
                  w
                </em>

                <em>
                  UAS-dco / w; en-GAL4 / +
                </em>
              </td>
            </tr>
            <tr>
              <td>
                <a
                  href="https://elifesciences.org/articles/45107/figures#fig7s3"
                >
                  Figure 7—figure supplement 3B, C
                </a>
              </td>
              <td>
                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18; pCaSpeR-dsh-GFP / +
                </em>

                <em>
                  y w dsh
                  <sup>
                    V26
                  </sup>
                  FRT18; pCaSpeR-dshST8-GFP / +
                </em>
              </td>
            </tr>
          </tbody>
        </table>
      </figure>

      <h3>
        Adult wing preparations
      </h3>

      <p>
        Adult wings were dehydrated in isopropanol and mounted in GMM (50%
        methyl salicylate, 50% Canada Balsam), and incubated overnight on a 60°C
        hot plate to clear. Wings were photographed at 20x magnification. To
        quantify trichome swirling, ImageJ was used to draw around a region near
        the posterior cross vein in which trichomes deviated significantly away
        from their normal orientation. Data were compared using ANOVA with
        Dunnett’s multiple comparisons test.
      </p>

      <h3>
        Immunolabelling
      </h3>

      <p>
        Pupal wings were dissected at 28 hr after puparium formation (APF) at
        25°C. Briefly, pupae were removed from their pupal case and fixed for
        25–60 min in 4% paraformaldehyde in PBS, depending on antibody
        combinations. Wings were then dissected and the outer cuticle removed,
        and were blocked for 1 hr in PBS containing 0.2% Triton X100 (PTX) and
        10% normal goat serum. Primary and secondary antibodies were incubated
        overnight at 4°C in PTX with 10% normal goat serum, and all washes were
        in PTX. After immunolabelling, wings were post-fixed in 4%
        paraformaldehyde in PBS for 30 min. Wings were mounted in 25 µl of PBS
        containing 10% glycerol and 2.5% DABCO, pH7.5. Wings expressing
        mApple-tagged proteins were mounted in 12.5 µl Vectashield, as this
        preserved the fluorescence for longer.
      </p>
      <p>
        Primary antibodies for immunolabelling were rabbit anti-Stbm (
        <a href="#bib54">
          Warrington et al., 2013
        </a>
        ), rat anti-Stbm (
        <a href="#bib51">
          Strutt and Strutt, 2008
        </a>
        ), affinity purified rabbit anti-Fz (
        <a href="#bib7">
          Bastock and Strutt, 2007
        </a>
        ), affinity purified rat anti-Pk (
        <a href="#bib47">
          Strutt et al., 2013a
        </a>
        ), rat anti-Dsh (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ), mouse monoclonal anti-Fmi (DSHB,
        <a href="#bib53">
          Usui et al., 1999
        </a>
        ), rabbit anti-GFP (Abcam cat#6556), mouse monoclonal anti-ß-gal 40-1a
        (DSHB) and rabbit anti-ß-gal (Cappel).
      </p>

      <h3>
        Western blotting
      </h3>

      <p>
        For pupal wing westerns, 28 hr APF pupal wings were dissected directly
        into sample buffer. One pupal wing equivalent was used per lane. For
        phosphatase treatments, 6 hr APF prepupal wing extracts were made in
        lysis buffer (50 mM Tris-HCl pH7.5, 150 mM NaCl, 0.5% Triton X-100, 1 x
        protease inhibitors [Roche]). Lysates were treated with 400 units lambda
        phosphatase (NEB) for 1 hr at 30°C, before addition of sample buffer.
      </p>
      <p>
        Western blots were probed with rabbit anti-Stbm (
        <a href="#bib39">
          Rawls and Wolff, 2003
        </a>
        ), affinity purified rabbit anti-Dsh (
        <a href="#bib45">
          Strutt et al., 2006
        </a>
        ), mouse monoclonal anti-Actin AC-40 (DSHB) and mouse monoclonal
        anti-α-Tubulin DM1-A (Sigma). SuperSignal West Dura Extended Duration
        Substrate (Thermo Scientific) was used for detection and a BioRad
        ChemiDoc XRS + was used for imaging. To quantitate total protein levels,
        intensities from three or four biological replicates were quantified
        using ImageJ. Data were compared using ANOVA with Tukey’s multiple
        comparisons test.
      </p>
      <p>
        For comparing levels of phosphorylated and unphosphorylated Dsh, bands
        on western blots migrated too close together to quantitate absolute band
        intensities. ImageJ was used to generate a band profile for each lane,
        and the maximum values of the phosphorylated and unphosphorylated bands
        were measured. Data is expressed as a ratio of this maximum intensity,
        and ratios from four biological replicate samples were compared using
        unpaired t-tests.
      </p>

      <h3>
        Imaging of fixed samples
      </h3>

      <p>
        Pupal wings were imaged on a Nikon A1R GaAsP confocal microscope using a
        60x NA1.4 apochromatic lens. Wings without clones were imaged posterior
        to vein 4; wings containing clones were imaged wherever they appeared in
        the wing. 9 Z-slices separated by 150 nm were imaged at a pixel size of
        70–80 nm, and the three brightest slices around apicolateral junctions
        were selected and averaged for each channel in ImageJ.
      </p>
      <p>
        Membrane masks and polarity nematics were generated in Packing Analyzer
        (
        <a href="#bib1">
          Aigouy et al., 2010
        </a>
        ). MATLAB scripts were used to calculate mean membrane intensity (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ). Polarity magnitude (maximum asymmetry ratio on a cell-by-cell basis)
        and variation in polarity angle were also calculated using MATLAB
        scripts (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ).
      </p>
      <p>
        To compare puncta between wild-type and mutant tissue, a MATLAB script
        was used to select puncta using the same threshold value in wild-type
        and mutant regions of the same wings (
        <a href="#bib50">
          Strutt et al., 2019
        </a>
        ). Puncta number per unit area was calculated, as well as mean puncta
        amount (puncta area multiplied by mean puncta intensity).
      </p>
      <p>
        Values for control and mutant regions of the same wings (for clones)
        were expressed as a ratio and compared using one sample t-tests; or were
        compared between images using unpaired t-tests or ANOVA for more than
        two genotypes. For all experiments n = number of wings.
      </p>

      <h3>
        Live imaging
      </h3>

      <p>
        For live imaging, a small piece of cuticle was removed from over the
        pupal wings of 28 hr APF pupae, and the exposed wing was mounted in a
        drop of Halocarbon 700 oil in a glass-bottomed dish. For FRAP analysis,
        images were 256 × 256 pixels, with a pixel size of 100 nm, and a pinhole
        of 1.2 AU. ‘Hub-and-spoke’ ROIs of 3–4 µm
        <sup>
          2
        </sup>
        were selected, that covered a vertex and three half-cell edges.
        Alternatively elliptical ROIs were selected to bleach puncta (ROIs 1–1.5
        µm
        <sup>
          2
        </sup>
        ) or entire junctions between two cells on a clone boundary (ROIs 2–2.5
        µm
        <sup>
          2
        </sup>
        ). Three pre-bleach images were taken at two frames/sec, and ROIs were
        then bleached using a 488 nm Argon laser at 80% with eight passes (1 s
        total time), which resulted in 60–75% bleaching. Immediately following
        bleaching, five images were taken at 5 s intervals, followed by 10
        images at 10 s intervals and 26 images at 15 s intervals. Laser power
        was adjusted to maintain constant power between different imaging
        sessions. If only EGFP was being imaged, a long pass GFP filter was
        used. If mRFP was present in the stock, EGFP was detected using a
        525–550 band pass filter.
      </p>

      <h3>
        FRAP processing
      </h3>

      <p>
        For data analysis, ImageJ was used to manually reselect up to six
        bleached regions in each image for each time point. The laser off
        background was subtracted, and the values were corrected for acquisition
        bleaching and normalised against the average of the prebleach values.
        Data were then plotted on an xy graph using Prism (v7 Graphpad),
        bleached regions within the same wing were averaged and a one-phase
        exponential curve was fitted for each wing. Multiple wings were then
        combined and an exponential association curve was fitted. An
        extra-sum-of-squares F test was used to compare curve plateaux (y[max]),
        and stable fractions were calculated as 1-y[max].
      </p>
      <p>
        For hub-and-spoke and boundary FRAP experiments, the stable amount of
        protein was calculated by measuring the intensity of the ROIs from the
        three pre-bleach images, and averaging per wing. The intensity was then
        corrected for distance from the coverslip as previously described (
        <a href="#bib49">
          Strutt et al., 2016
        </a>
        ), and this value was then multiplied by the stable fraction (1-y[max])
        for each wing. The stable amounts were then averaged across wings.
      </p>

      <p>
        Overall junctional intensities, and stable and unstable amounts were
        compared between genotypes using unpaired t-tests, or one-way ANOVA for
        more than two genotypes. Post-hoc tests were used to compare individual
        samples: Dunnett’s multiple comparison test was used to compare the
        control to the rest of the genotypes in the experiment; Tukey-Kramer’s
        multiple comparison test to compare all genotypes within an experiment;
        and Holm-Šídák’s multiple comparison test was used to compare genotypes
        pair-wise.
      </p>

      <p>
        Each experiment was performed on multiple wings from different pupae,
        which represent biological replicates (n = number of wings). For each
        wing, 4 ROIs were selected for FRAP analysis, and these were treated as
        technical replicates and were averaged per wing to produce a y[max] and
        a stable amount per wing.
      </p>
      <p>
        Based on the mean intensity and standard deviation of a control set of
        wings, we calculated that a sample size of 6 wings per genotype would
        allow detection of differences of 20% in the means, in a pair-wise
        comparison, with a power of 0.8 and α 0.05 (using G*Power). As standard
        deviations were larger for some genotypes, we aimed for 10 wings per
        genotype. Data was excluded if the ROI recovery curve failed the
        ‘replicates test for lack of fit’ in GraphPad Prism, or if the wing
        moved out of focus during the course of imaging.
      </p>

      <h2>
        References
      </h2>

      <ol itemtype="https://stencila.github.io/schema/CiteGroup">
        <li>
          <ul itemtype="https://stencila.github.io/schema/Cite">
            <div itemtype="https://schema.org/CreativeWork">
              <a
                href="https://doi.org/10.1016/j.cell.2010.07.042"
                itemprop="title"
                >Cell flow reorients the axis of planar polarity in the wing
                epithelium of Drosophila</a
              >

              <ol itemprop="authors">
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:B+Aigouy%22"
                    >B Aigouy</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:R+Farhadifar%22"
                    >R Farhadifar</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:DB+Staple%22"
                    >DB Staple</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:A+Sagner%22"
                    >A Sagner</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:JC+R%C3%B6per%22"
                    >JC Röper</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:F+J%C3%BClicher%22"
                    >F Jülicher</a
                  >
                </li>
                <li itemtype="https://schema.org/Person">
                  <a
                    href="https://scholar.google.com/scholar?q=%22author:S+Eaton%22"
                    >S Eaton</a
                  >
                </li>
              </ol>

              <span itemprop="datePublished">(2010)</span>

              <ul itemtype="https://stencila.github.io/schema/PublicationIssue">
                <em itemprop="title">Cell</em>
                <strong itemprop="issueNumber">142</strong>
                <span itemprop="pagination">:773–786.</span>
              </ul>

              <a
                href="https://doi.org/10.1016/j.cell.2010.07.042"
                itemprop="url"
                >https://doi.org/10.1016/j.cell.2010.07.042</a
              >

              <ul itemprop="abstracts">
                <li itemtype="https://stencila.github.io/schema/Abstract">
                  <a href="https://www.ncbi.nlm.nih.gov/pubmed/20813263"
                    >PubMed</a
                  >
                </li>
                <li itemtype="https://stencila.github.io/schema/Abstract">
                  <a
                    href="https://scholar.google.com/scholar_lookup?title=Cell+flow+reorients+the+axis+of+planar+polarity+in+the+wing+epithelium+of+Drosophila&amp;author=B+Aigouy&amp;author=R+Farhadifar&amp;author%5B2%5D=DB+Staple&amp;author%5B3%5D=A+Sagner&amp;author%5B4%5D=JC+R%C3%B6per&amp;author%5B5%5D=F+J%C3%BClicher&amp;author%5B6%5D=S+Eaton&amp;publication_year=2010&amp;journal=Cell&amp;volume=142&amp;pages=pp.+773%E2%80%93786&amp;pmid=20813263"
                    >Google Scholar</a
                  >
                </li>
              </ul>
            </div>
          </ul>
        </li>
        <li>
          2
        </li>
        <li>
          3
        </li>
        <li>
          4
        </li>
        <li>
          5
        </li>
        <li>
          6
        </li>
        <li>
          7
        </li>
        <li>
          8
        </li>
        <li>
          9
        </li>
        <li>
          10
        </li>
        <li>
          11
        </li>
        <li>
          12
        </li>
        <li>
          13
        </li>
        <li>
          14
        </li>
        <li>
          15
        </li>
        <li>
          16
        </li>
        <li>
          17
        </li>
        <li>
          18
        </li>
        <li>
          19
        </li>
        <li>
          20
        </li>
        <li>
          21
        </li>
        <li>
          22
        </li>
        <li>
          23
        </li>
        <li>
          24
        </li>
        <li>
          25
        </li>
        <li>
          26
        </li>
        <li>
          27
        </li>
        <li>
          28
        </li>
        <li>
          29
        </li>
        <li>
          30
        </li>
        <li>
          31
        </li>
        <li>
          32
        </li>
        <li>
          33
        </li>
        <li>
          34
        </li>
        <li>
          35
        </li>
        <li>
          36
        </li>
        <li>
          37
        </li>
        <li>
          38
        </li>
        <li>
          39
        </li>
        <li>
          40
        </li>
        <li>
          41
        </li>
        <li>
          42
        </li>
        <li>
          43
        </li>
        <li>
          44
        </li>
        <li>
          45
        </li>
        <li>
          46
        </li>
        <li>
          47
        </li>
        <li>
          48
        </li>
        <li>
          49
        </li>
        <li>
          50
        </li>
        <li>
          51
        </li>
        <li>
          52
        </li>
        <li>
          53
        </li>
        <li>
          54
        </li>
        <li>
          55
        </li>
        <li>
          56
        </li>
        <li>
          57
        </li>
        <li>
          58
        </li>
      </ol>

      <h2>
        Decision letter
      </h2>

      <p>
        Thank you for submitting your article "Reciprocal action of Casein
        Kinase Iε on core planar polarity proteins regulates clustering and
        asymmetric localisation" for consideration by
        <em>
          eLife
        </em>
        . Your article has been reviewed by three peer reviewers, and the
        evaluation has been overseen by a Reviewing Editor and Utpal Banerjee as
        the Senior Editor. The reviewers have opted to remain anonymous.
      </p>
      <p>
        As you will see, all of the reviewers were impressed with the importance
        and novelty of your work. I am including the three reviews at the end of
        this letter, as there are a variety of specific and useful suggestions
        in them. We appreciate that the reviewers' comments cover a broad range
        of suggestions for improving the manuscript. We look forward to
        receiving your revised manuscript.
      </p>
      <p>
        <em>
          Reviewer #1:
        </em>
      </p>
      <p>
        The manuscript examines the role the Casein kinase Dco plays in the
        regulation of core PCP proteins, especially Stbm and Dsh. The study
        confirms recent findings the Dco affects Stbm phosphorylation, and
        extends these by examining the effects of phosphorylation site mutants
        on
        <em>
          Drosophila
        </em>
        wing PCP, and on Stbm polarization, stability at junctions, puncta size,
        etc. Thus they go well beyond the study of Kelly et al., 2016 especially
        in vivo, and in detail some of their results disagree. Intriguingly,
        phosphomutant and phosphomimetic Stbm both have abnormal and even
        dominant negative effects on PCP and are associated in decreased and
        increased junctional stability, respectively. The authors do a nice job
        incorporating this into their model.
      </p>
      <p>
        Previous work had suggested that Stbm phosphorylation could be regulated
        by Dsh and Fz. However, the authors find little evidence of this in vivo
        from fz and dsh mutants. Instead, they find a strong effect from Pk.
        While they do not have a firm understanding of how this works, they
        provide some persuasive arguments that it is likely fairly direct,
        rather than through simple assembly of the PCP complex or through Fz or
        Dsh.
      </p>
      <p>
        Finally, they go on to show that Dco also has an independent effect on
        Dsh levels and phosphorylation.
      </p>
      <p>
        While this paper lacks the zing of recent studies on polarity-inducing
        feedback mechanisms, it is certainly a solid study that extends previous
        work on the role of PCP component phosphorylation. I think it deserves
        publication.
      </p>
      <p>
        <em>
          Reviewer #2:
        </em>
      </p>
      <p>
        In this study, Strutt et al. investigate the role of phosphorylation in
        the establishment of planar cell polarity (PCP) in
        <em>
          Drosophila
        </em>
        . Using immunofluorescence, live imaging, FRAP, biochemistry, and
        genetic mosaic approaches, they provide evidence that Stbm is less
        stable and more mobile when phosphorylated, and conversely, more stable
        when unphosphorylated. In contrast to Stbm, Dsh was found to be more
        stable when phosphorylated. They also provide evidence that Pk
        negatively regulates Stbm phosphorylation. Their findings suggest that
        Casein Kinase Ie mediates Stbm phosphorylation, confirming other
        reports, and extend this finding by providing genetic evidence that
        Casein Kinase Ie phosphorylates both Stbm and Dsh independently. They
        propose a model for how modifying differential protein stability could
        be means to regulate sorting of PCP proteins.
      </p>
      <p>
        Mechanisms that regulate the establishment and maintenance of PCP is of
        high interest to the fields of polarity and developmental biology.
        Although Stbm/Vang and Dsh/Dvl have been known to be phosphorylated for
        some time, and that their phosphorylation is important for PCP, it was
        not known how phosphorylation regulates PCP protein dynamics and
        function. By providing further support that Casein Kinase Ie-mediates
        phosphorylation of Stbm and Dsh and by demonstrating that
        phosphorylation alters PCP protein stability and sorting, this study
        provides important and substantial contributions to the field. A
        strength of the study is the use of tissue extracts from transgenic and
        mutant lines to measure phosphorylation states (as opposed to
        transfected cells), but the reliance on very subtle band shifts as the
        sole means to detect phosphorylation is a weakness. The manuscript text
        is generally well written, however the data and figures could have been
        presented more clearly. While statistics used were appropriate and
        thorough, the quantification is often confusing and needs to be made
        more transparent. Though the findings are significant and impactful with
        some mechanistic detail, there are concerns that the work does not go
        far enough to extend current knowledge and has limited accessibility in
        its current state for the broad readership of
        <em>
          eLife
        </em>
        .
      </p>
      <p>
        Major comments:
      </p>
      <p>
        - Inclusion of a table of
        <em>
          Drosophila
        </em>
        genotypes used with direct reference of how they are labeled/referred to
        in figures their phenotypes would be very helpful.
      </p>
      <p>
        - No biochemical evidence is shown to support that direct binding of Pk
        to Stbm protects Stbm from phosphorylation (as discussed in the
        Discussion, fourth paragraph).
      </p>
      <p>
        - Figure 1 lays the foundation for all subsequent figures and could be
        presented more clearly for a broader audience. For example, the
        schematic in A makes sense to a PCP specialist but needs more context
        for someone less familiar with the field. Zoomed images for E-F will
        help better visualize trichome orientation, and a reorganization of E-M
        to better pair panels of the same genotypes would help. Additionally, an
        overlay of polarity vectors for I-J would be helpful as Stbm asymmetry
        is subtle and not apparent to the untrained eye.
      </p>
      <p>
        - The authors state that "only small puncta were observed in Stbm
        phosphomimetic flies, and non-puncta material also increased (Figure
        1J). However, in the Stbm phosphomutant, Stbm still clustered into large
        puncta, despite the loss of asymmetry (Figure 1I)." Quantification of
        the puncta sizes in Figure 1I-J and Figure 2A-D are lacking and
        necessary as many of the future conclusions rely on these observations.
      </p>
      <p>
        - Figure 2G-H is a critical piece of data to support the authors model
        that -P promotes sorting of opposite PCP complexes, and should be
        presented more convincingly. Panel H shows Diego localized to both sides
        of cell junctions in Stbm phosphomutants, suggesting PCP complexes of
        opposite orientations are mixed. According to the model, puncta in wild
        type flies represent sorted PCP complexes, but is this observed during
        intermediate stages of polarization? Diego is clearly segregated in the
        fully polarized control in panel G, but at earlier stages, do they see
        puncta along a single junction that are sorted in opposite orientations?
        This control would strengthen the argument that it is the sorting of
        opposite complexes within puncta that is defective in their mutants.
        According to the model, stbmS(AII)E complexes/puncta should be sorted
        and thus, that data should be included stbmS(AII)E phosphomimetic for
        G-H panels.
      </p>
      <p>
        - Figure 3. Stable to unstable amounts should be displayed separately to
        better see the changes, it is hard to compare the relative amounts as
        currently displayed (this holds true for all FRAP data).
      </p>
      <p>
        - Figure 5. The western blot bands in A demonstrating a shift in Stbm
        are not obvious. This is main evidence that Pk influences Stbm
        phosphorylation, but it's not wholly convincing. In general, detecting
        changes in -P solely by band shifts is a general weakness of the overall
        approach since these differences are very subtle, hard to quantify, and
        open to other interpretations. Although this is probably beyond scope of
        this paper, generating a phospho-specific Ab or performing mass spec
        analyses of -P sites would go a long way towards strengthening the
        results and lending support to the overall model.
      </p>
      <p>
        - The model in Figure 7 would benefit from inclusion of Dco. In
        addition, do the authors think that Pk destabilizes Fz via inhibiting
        Dco-dependent -P of Dsh?
      </p>
      <p>
        <em>
          Reviewer #3:
        </em>
      </p>
      <p>
        The manuscript by Strutt and colleagues addresses the mechanisms by
        which protein complexes become asymmetrically distributed on cell
        junctions, a process important for establishing planar cell polarity.
        The authors identified an important role of CK1 in phosphorylating core
        proteins and thereby regulating their clustering/membrane association.
        This work gives interesting new insights into the role that
        phosphorylation plays in establishing the asymmetry of core proteins at
        cell junctions. The manuscript is well written and the data are
        convincing for the most part. I have a few concerns, which I expect
        could be addressed within a reasonable time by the authors:
      </p>
      <p>
        Figure 1J, L The authors claim that in Stbm phosphomimetic flies only
        small puncta are observed, whereas in Stbm phosphomutant flies, Stbm
        clusters in large puncta. The puncta are not obvious from these images.
        Higher magnification images should be shown and puncta size needs to be
        quantified. Moreover, the authors should mention whether
        <em>
          stbm
        </em>
        [6] mutants still express Stbm protein, i.e. do these stainings show
        Stbm[6] and phosphomutant Stbm or only the phosphomutant Stbm?
      </p>
      <p>
        Figure 2G, H I can see that in the Stbm phosphomutant flies Diego forms
        "non-polarized" puncta, however, I do not see evidence that these puncta
        co-localize with Stbm, as the authors claim.
      </p>
      <p>
        Figure 3A. The authors should briefly explain the concept of the
        'hub-and-spoke' FRAP methods, so that readers do not have to look up the
        reference in order to understand the experiment. Moreover, images of the
        FRAP experiment should be shown (e.g. in the supplement) to allow
        readers to evaluate the data.
      </p>
      <p>
        Figure 3 FRAP analysis allows to estimate the mobile fraction of a
        protein, and the authors use this method to show that phosphorylation of
        Stbm increases its mobile fraction. It is however, less clear how the
        authors infer from the FRAP analysis that phosphorylation suppresses
        clustering in complexes. While complex formation might be one mechanism
        to reduce the mobile fraction of a protein, it is not the only one. The
        authors should elaborate on this issue.
      </p>
      <p>
        <a href="https://doi.org/10.7554/eLife.45107.033">
          https://doi.org/10.7554/eLife.45107.033
        </a>
      </p>

      <h2>
        Author response
      </h2>

      <blockquote>
        <p>
          Reviewer #2:
        </p>
        <p>
          […] A strength of the study is the use of tissue extracts from
          transgenic and mutant lines to measure phosphorylation states (as
          opposed to transfected cells), but the reliance on very subtle band
          shifts as the sole means to detect phosphorylation is a weakness. The
          manuscript text is generally well written, however the data and
          figures could have been presented more clearly. While statistics used
          were appropriate and thorough, the quantification is often confusing
          and needs to be made more transparent. Though the findings are
          significant and impactful with some mechanistic detail, there are
          concerns that the work does not go far enough to extend current
          knowledge and has limited accessibility in its current state for the
          broad readership of eLife.
        </p>
        <p>
          Major comments:
        </p>
        <p>
          - Inclusion of a table of
          <em>
            Drosophila
          </em>
          genotypes used with direct reference of how they are labeled/referred
          to in figures their phenotypes would be very helpful.
        </p>
      </blockquote>
      <p>
        We have added a table of all genotypes associated with each figure.
      </p>
      <blockquote>
        <p>
          - No biochemical evidence is shown to support that direct binding of
          Pk to Stbm protects Stbm from phosphorylation (as discussed in the
          Discussion, fourth paragraph).
        </p>
      </blockquote>
      <p>
        We have discussed other possible models for how Pk might regulate Stbm
        phosphorylation in the manuscript, but we found evidence arguing against
        the other models that we considered. We agree however that we don't have
        direct evidence for the idea that binding of Pk to Stbm protects Stbm
        from phosphorylation; this is why we put the associated model figure in
        the supplementary data rather than in the main manuscript. However we
        think it is useful to suggest hypotheses about the mechanism, as this is
        then a basis for future studies. In the future we will be looking for
        direct evidence for this model, but such experiments are beyond the
        scope of the current manuscript.
      </p>
      <blockquote>
        <p>
          - Figure 1 lays the foundation for all subsequent figures and could be
          presented more clearly for a broader audience. For example, the
          schematic in A makes sense to a PCP specialist but needs more context
          for someone less familiar with the field. Zoomed images for E-F will
          help better visualize trichome orientation, and a reorganization of
          E-M to better pair panels of the same genotypes would help.
          Additionally, an overlay of polarity vectors for I-J would be helpful
          as Stbm asymmetry is subtle and not apparent to the untrained eye.
        </p>
      </blockquote>
      <p>
        These are all good points, and we have added extra introductory panels
        to the first figure. The adult wings in (now in Figure 2 and Figure
        2—figure supplement 1) have been replaced with higher magnification
        images. We have also overlaid polarity vectors on the images in Figure 2
        and Figure 2 supplements 1 and 2. Finally, we have also included higher
        magnification images of puncta in these genotypes, in response to
        reviewer 3.
      </p>
      <blockquote>
        <p>
          - The authors state that "only small puncta were observed in Stbm
          phosphomimetic flies, and non-puncta material also increased (Figure
          1J). However, in the Stbm phosphomutant, Stbm still clustered into
          large puncta, despite the loss of asymmetry (Figure 1I)."
          Quantification of the puncta sizes in Figure 1I-J and Figure 2A-D are
          lacking and necessary as many of the future conclusions rely on these
          observations.
        </p>
      </blockquote>

      <p>
        We have now included measurements of puncta number and size in Figure 2.
        We previously attempted to threshold images using Stbm immunolabelling,
        using the same threshold for wild-type and mutant regions of the same
        wing. However as overall junctional levels of Stbm increase in both the
        phosphomutant and the phosphomimetic, this was problematic and resulted
        in apparent increased values in both cases. We have now got round this
        issue by using Fmi immunolabelling to threshold the images: as Fmi
        co-localises with Stbm and junctional levels of Fmi do not change in the
        phosphomutants, this successfully identifies puncta without "spreading"
        into neighbouring non-puncta regions. The major finding is that puncta
        <em>
          number
        </em>
        is similar in wild-type and phosphomutant; suggesting a similar degree
        of "punctateness"; while the phosphomimetic is less punctate. The amount
        of Stbm in puncta (puncta area multiplied by puncta intensity) is
        however slightly lower in both the phosphomutant and phosphomimetic. We
        think the smaller puncta size in phosphomutant and phosphomimetic may be
        due to the fact that negative feedback limits the growth of individual
        puncta (as seen in our FRAP experiments using Fmi-EGFP and Fz-EGFP).
      </p>
      <p>
        We have modified the manuscript to take into account these
        quantitations; in particular we have stated that the phosphomimetic
        forms fewer puncta rather than that it forms only small puncta.
      </p>
      <p>
        However, the main conclusion still stands: that the Stbm phosphomimetic
        clusters into puncta more readily than the phosphomimetic and thus the
        coupling between puncta formation and asymmetry is lost.
      </p>
      <blockquote>
        <p>
          - Figure 2G-H is a critical piece of data to support the authors model
          that -P promotes sorting of opposite PCP complexes, and should be
          presented more convincingly. Panel H shows Diego localized to both
          sides of cell junctions in Stbm phosphomutants, suggesting PCP
          complexes of opposite orientations are mixed. According to the model,
          puncta in wild type flies represent sorted PCP complexes, but is this
          observed during intermediate stages of polarization? Diego is clearly
          segregated in the fully polarized control in panel G, but at earlier
          stages, do they see puncta along a single junction that are sorted in
          opposite orientations? This control would strengthen the argument that
          it is the sorting of opposite complexes within puncta that is
          defective in their mutants. According to the model, stbmS(AII)E
          complexes/puncta should be sorted and thus, that data should be
          included stbmS(AII)E phosphomimetic for G-H panels.
        </p>
      </blockquote>
      <p>
        We have not looked at Dgo localisation at earlier developmental stages,
        but we speculate that the sorting would be less good than at 28 hr APF.
        We expect that sorting
        <em>
          within
        </em>
        puncta would be reduced; and that most puncta would contain complexes in
        both orientations rather than seeing puncta along a single junction that
        are sorted in opposite orientations.
      </p>

      <p>
        We also don't predict that individual Stbm phosphomimetic
        complexes/puncta should be sorted. Our data suggest that Stbm
        phosphomimetics behave like
        <em>
          pk
        </em>
        mutants, in which Stbm is hyperphosphorylated. In our model
        hyperphosphorylation promotes Stbm mobility and thus the puncta that do
        form are unstable. This means that complexes are continually forming and
        unforming and that complexes of the same orientation fail to accumulate.
        It would be interesting to test this directly, but we are unable to do
        the twin clone experiment with our current reagents, as
        <em>
          P[acman]-stbmS[All]E, P[acman]-mApple-dgo
        </em>
        and
        <em>
          P[acman]-EGFP-dgo
        </em>
        are in the same
        <em>
          attP
        </em>
        site.
      </p>

      <blockquote>
        <p>
          - Figure 3. Stable to unstable amounts should be displayed separately
          to better see the changes, it is hard to compare the relative amounts
          as currently displayed (this holds true for all FRAP data).
        </p>
      </blockquote>
      <p>
        We agree that it is not easy to compare the unstable amounts in the
        current graphs. However, if we plot stable and unstable amounts
        separately, we will "lose" the other important data regarding total
        intensity. We have however added extra supplementary figures for all
        FRAP data, showing the total intensity, stable amount and unstable
        amounts as separate plots.
      </p>
      <blockquote>
        <p>
          - Figure 5. The western blot bands in A demonstrating a shift in Stbm
          are not obvious. This is main evidence that Pk influences Stbm
          phosphorylation, but it's not wholly convincing. In general, detecting
          changes in -P solely by band shifts is a general weakness of the
          overall approach since these differences are very subtle, hard to
          quantify, and open to other interpretations. Although this is probably
          beyond scope of this paper, generating a phospho-specific Ab or
          performing mass spec analyses of -P sites would go a long way towards
          strengthening the results and lending support to the overall model.
        </p>
      </blockquote>
      <p>
        We agree that the band shifts are subtle, and that is why we have shown
        duplicates of the critical samples. We have now also included an extra
        supplementary figure panel (Figure 6—figure supplement 1A) in response
        to reviewer 1, to show that the decreased migration of Stbm in
        <em>
          pk
        </em>
        mutants is sensitive to phosphatase.
      </p>
      <p>
        We also agree that phospho-specific antibodies would be nice, but also
        that this is beyond the scope of this paper. Furthermore, we note that
        other researchers have made phospho-specific antibodies (e.g. for Dsh,
        Yanfeng et al., 2011) and it is difficult to get enough sensitivity on
        westerns in an in vivo context.
      </p>
      <blockquote>
        <p>
          - The model in Figure 7 would benefit from inclusion of Dco. In
          addition, do the authors think that Pk destabilizes Fz via inhibiting
          Dco-dependent -P of Dsh?
        </p>
      </blockquote>
      <p>
        We have added Dco to the model diagram (now in Figure 8).
      </p>
      <p>
        In our previous work, we proposed two mechanisms by which Pk could
        destabilise Fz. In the first, Dsh would normally protect Fz from
        endocytosis (e.g. by promoting multimerisation); and that Pk binding to
        Dsh could block this protective function. In the second mechanism, we
        proposed that binding of Pk to Fz/Dsh could lead to recruitment of an
        enzyme that mediates a change in post-translational modification of Dsh
        or Fz. In either model, we could envisage either a direct role for Dco
        (e.g. Pk locally inhibits Dco function or promotes a phosphatase that
        opposes Dco activity) or indirect roles (e.g. multimerised Dsh is a
        better target for Dco or is protected from a phosphatase, and Pk opposes
        multimerisation thus indirectly leading to a change in the Dco-mediated
        phosphorylation of Dsh). However, we think we might be speculating
        beyond what the data can really support if we included such ideas in the
        manuscript.
      </p>
      <blockquote>
        <p>
          Reviewer #3:
        </p>
        <p>
          […] I have a few concerns, which I expect could be addressed within a
          reasonable time by the authors:
        </p>
        <p>
          Figure 1J, L The authors claim that in Stbm phosphomimetic flies only
          small puncta are observed, whereas in Stbm phosphomutant flies, Stbm
          clusters in large puncta. The puncta are not obvious from these
          images. Higher magnification images should be shown and puncta size
          needs to be quantified. Moreover, the authors should mention whether
          stbm[6] mutants still express Stbm protein, i.e. do these stainings
          show Stbm[6] and phosphomutant Stbm or only the phosphomutant Stbm?
        </p>
      </blockquote>
      <p>
        We have now shown higher magnification images of the puncta in Figure 2
        and Figure 2—figure supplements 1 and 2. We have also provided
        quantitations of puncta number and size (see response to reviewer 2).
      </p>
      <p>
        We have also clarified in the manuscript that the clones of the
        phosphorylation site mutants are generated in the absence of endogenous
        Stbm.
      </p>
      <blockquote>
        <p>
          Figure 2G, H, I can see that in the Stbm phosphomutant flies Diego
          forms "non-polarized" puncta, however, I do not see evidence that
          these puncta co-localize with Stbm, as the authors claim.
        </p>
      </blockquote>
      <p>
        We have now shown in Figure 3—figure supplement 1 that EGFP-Dgo
        co-localises in puncta with the other core proteins, in both a wild-type
        and a Stbm phosphomutant background.
      </p>
      <blockquote>
        <p>
          Figure 3A. The authors should briefly explain the concept of the
          'hub-and-spoke' FRAP methods, so that readers do not have to look up
          the reference in order to understand the experiment. Moreover, images
          of the FRAP experiment should be shown (e.g. in the supplement) to
          allow readers to evaluate the data.
        </p>
      </blockquote>
      <p>
        We have added more explanation to the text to better explain the
        methodology. We have also added some example FRAP images (pre-bleach,
        immediately post-bleach and after 500 sec recovery) for the first
        experiment, and the entire FRAP datasets are in the source data.
      </p>
      <blockquote>
        <p>
          Figure 3 FRAP analysis allows to estimate the mobile fraction of a
          protein, and the authors use this method to show that phosphorylation
          of Stbm increases its mobile fraction. It is however, less clear how
          the authors infer from the FRAP analysis that phosphorylation
          suppresses clustering in complexes. While complex formation might be
          one mechanism to reduce the mobile fraction of a protein, it is not
          the only one. The authors should elaborate on this issue.
        </p>
      </blockquote>
      <p>
        We agree that in general a reduction in the mobile fraction of a protein
        can be caused by mechanisms other than complex formation; for instance
        other explanations could be reduced vesicular trafficking or tethering
        to another structure such as the cytoskeleton. However, our model is
        based not only on the FRAP data but on the fact that we see changes in
        cluster formation (puncta) in our images (now quantitated in the revised
        manuscript). Taken together, we think our current working model is the
        most plausible. We would prefer not to add further speculation to the
        Discussion, which is already quite long.
      </p>
      <p>
        <a href="https://doi.org/10.7554/eLife.45107.034">
          https://doi.org/10.7554/eLife.45107.034
        </a>
      </p>

      <h2>
        Article and author information
      </h2>

      <h3>
        Author details
      </h3>
      <ol>
        <li>
          <p></p>
          <h4>
            Jessica Gamage
          </h4>

          Department of Biomedical Science, University of Sheffield, Sheffield,
          United Kingdom

          <h5>
            Contribution
          </h5>
          Investigation, Writing—review and editing

          <h5>
            Competing interests
          </h5>
          No competing interests declared

          <p></p>
        </li>
      </ol>

      <h3>
        Funding
      </h3>

      <h4>
        Wellcome Trust (100986/Z/13/Z)
      </h4>

      <ul>
        <li>
          Helen Strutt
        </li>
        <li>
          David Strutt
        </li>
      </ul>

      <h4>
        Medical Research Council (G0900203-1/1)
      </h4>

      <h4>
        Wellcome Trust (210630/Z/18/Z)
      </h4>

      <ul>
        <li>
          Helen Strutt
        </li>
        <li>
          David Strutt
        </li>
      </ul>

      <h4>
        Medical Research Council (G1000405-1/1)
      </h4>

      <p>
        The funders had no role in study design, data collection and
        interpretation, or the decision to submit the work for publication.
      </p>

      <h3>
        Acknowledgements
      </h3>

      <p>
        We thank the Bloomington Drosophila Stock Center for fly stocks, Tanya
        Wolff and the Developmental Studies Hybridoma Bank for antibodies, and
        Genetivision for generating transgenics. Natalia Bulgakova, Katie
        Fisher, Simon Fellgett and Melissa Gammons are thanked for comments on
        the manuscript, and the fly room staff for excellent technical support.
        Imaging was performed in the Wolfson Light Microscopy Facility. The work
        was funded by Wellcome Senior Fellowship awards to DS [100986/Z/13/Z and
        210630/Z/18/Z] and an MRC studentship (grant numbers G0900203-1/1 and
        G1000405-1/1).
      </p>

      <h3>
        Senior Editor
      </h3>

      <ol>
        <li>
          Utpal Banerjee, University of California, Los Angeles, United States
        </li>
      </ol>

      <h3>
        Reviewing Editor
      </h3>

      <ol>
        <li>
          Jeremy Nathans, Johns Hopkins University School of Medicine, United
          States
        </li>
      </ol>

      <h3>
        Publication history
      </h3>

      <ol>
        <li>
          Received: January 11, 2019
        </li>
        <li>
          Accepted: May 14, 2019
        </li>
        <li>
          Accepted Manuscript published:
          <a href="https://elifesciences.org/articles/45107v1">
            May 15, 2019 (version 1)
          </a>
        </li>
        <li>
          Version of Record published:
          <a href="https://elifesciences.org/articles/45107">
            May 30, 2019 (version 2)
          </a>
        </li>
      </ol>

      <h3>
        Copyright
      </h3>

      <p>
        © 2019, Strutt et al.
      </p>
      <p>
        This article is distributed under the terms of the
        <a href="http://creativecommons.org/licenses/by/4.0/">
          Creative Commons Attribution License
        </a>
        , which permits unrestricted use and redistribution provided that the
        original author and source are credited.
      </p>
    </article>

    <script src="../themes/eLife/index.ts" async></script>
  </body>
</html>
